<html>

<head>

  <title>I-TASSER results</title>

</head>



<script type="text/javascript" src="../../bin/jmol/Jmol.js"></script>

<body bgcolor="#FFFFFF">

<font face="Times New Roman">

[<a href="http://zhanglab.ccmb.med.umich.edu">Home</a>]

[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER">Server</a>]

[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/queue.php">Queue</a>]

[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/about.html">About</a>]

[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/remove.html">Remove</a>]

[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/statistics.html">Statistics</a>]

<br/>

<table border="0" style="table-layout:fixed;" width="100%" cellpadding="2" cellspaceing="0">

<tr><td>

  <h1 align="center">I-TASSER results for job id S185852</h1>

<h3 align="center"> [Click on <a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S185852/S185852_results.tar.bz2">S185852_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page] </h3>

</td></tr>

<tr><td>

   <div style="background:#6599FF;width:200px;">

   <table><tr><td style= "valign:middle;">

   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Submitted Sequence</font>

   </td></tr></table>

   </div>

   <br/>

   <div style="position:relative;left:30px;">

   <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">

   <tr><td colspan="2">>protein<br/>MSYVPHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEI<br/>NDERIREALANIEKNGVTGRASIVKGNFFEVDISEATVVTMFLLTNVNEMLKPKLEKELK<br/>PGTRVVSHEFEIRGWNPKEVIKVEDGNMNHTVYLYVIGEHKLE<br/></td></tr>

   </table>

   </div>

</td></tr>

  <tr><td>

   <br/>

   <div style="background:#6599FF;width:300px;">

   <table><tr><td valign="middle">

   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Secondary Structure</font>

   </td></tr></table>

   </div>

   <br/>

   <div style="position:relative;left:30px;">

    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">

    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;<br/>MSYVPHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGRASIVKGNFFEVDISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEIRGWNPKEVIKVEDGNMNHTVYLYVIGEHKLE</td></tr>

    <tr><td><font="Arial"><span title="Secondary Structure Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">CCCCCCCCCCCCC<font color="red">HHHHHHHHHH</font>CCCCCCC<font color="blue">SSSSSS</font>CCCC<font color="red">HHHHHHHHH</font>CCCC<font color="blue">SSSSSS</font>CC<font color="red">HHHHHHHHHHHHH</font>CCCCCC<font color="blue">SSSSS</font>CC<font color="red">HHH</font>CCCCCCC<font color="blue">SSSSS</font>CCCC<font color="red">HHHHHHHHHHHH</font>CCCCC<font color="blue">SSSSSS</font>CCCCCCC<font color="red">HHH</font><font color="blue">SSSS</font>CCCCCCC<font color="blue">SSSSSS</font>CC<font color="red">HHH</font>CC</td>

    <tr><td><font="Arial"><span title="Confidence Score of Prediction"><b>Conf.Score</b></span></font></td><td BGCOLOR="#ECECEC">9878999823582999999997518998699999617788999999997799889999999999999999999819998779997997780258987999945722789999999956186859999864689998141788216777635999988803158</td></tr>

    </table>

  </div>

  </td></tr>

  <tr><td>

   <br/>

   <div style="background:#6599FF;width:300px;">

   <table><tr><td valign="middle">

   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Solvent Accessibility</font>

   </td></tr></table>

   </div>

   <br/>

   <div style="position:relative;left:30px;">

    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">

    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;<br/>MSYVPHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGRASIVKGNFFEVDISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEIRGWNPKEVIKVEDGNMNHTVYLYVIGEHKLE</td></tr>

    <tr><td><font="Arial"><span title="Solvent Accessibility Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">7434141131203450032006217166622000031330200010034243330000020430042024004425145303022320351334601000000114204401630362037402000011314714264224045563522000031356548</td><tr>

    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC">Values range from <b>0</b> (buried residue) to <b>9</b> (highly exposed residue)</td><tr>

    </table>

  </div>

  </td></tr>

  <tr><td>

   <br/>

   <div style="background:#6599FF;width:300px;">

   <table><tr><td valign="middle">

   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Predicted normalized B-factor</font>

   </td></tr></table>

   </div>

   <br/>

   <div style="position:relative;left:30px;">

    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">

    <tr> <td align =center > <img src=BFP.png>  </td>  </tr>

    <tr> <td align =center >  <a href="lscore.txt" target="_blank"  >Read more about predicted normalized B-factor</a></td>  </tr>

    </table>

  </div>

  </td></tr>

  <tr><td>

   <br/>

   <div style="background:#6599FF;width:350px;">

   <table><tr><td style= "valign:middle;">

   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Top 10 templates used by I-TASSER</font>

   </td></tr></table>

   </div>

   <br/>

   <div style="position:relative;left:30px;">

   <table style="font-size:13px;background:#F2F2F2;">

      <tr> <th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br/>Hit</font></th><th align="center" valign="top"><font face="Arial">Iden1</font></th><th align="center" valign="top"><font face="Arial">Iden2</font></th><th valign="top"><font face="Arial">Cov.</font></th><th valign="top"><font face="Arial">Norm.<br/>Z-score</font></th><th valign="top"><font face="Arial">Download<br/>Align.</font></th><td>&nbsp;</td><td BGCOLOR="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;</td></tr>

      <tr><td colspan="7"></td><th align="right" valign ="bottom" ><font face="Arial">Sec.Str<br/>Seq</font></th><td BGCOLOR="#D7D7D7" style="font-family:Monospace;font-size:13px;">CCCCCCCCCCCCC<font color="red">HHHHHHHHHH</font>CCCCCCC<font color="blue">SSSSSS</font>CCCC<font color="red">HHHHHHHHH</font>CCCC<font color="blue">SSSSSS</font>CC<font color="red">HHHHHHHHHHHHH</font>CCCCCC<font color="blue">SSSSS</font>CC<font color="red">HHH</font>CCCCCCC<font color="blue">SSSSS</font>CCCC<font color="red">HHHHHHHHHHHH</font>CCCCC<font color="blue">SSSSSS</font>CCCCCCC<font color="red">HHH</font><font color="blue">SSSS</font>CCCCCCC<font color="blue">SSSSSS</font>CC<font color="red">HHH</font>CC<br/>MSYVPHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGRASIVKGNFFEVDISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEIRGWNPKEVIKVEDGNMNHTVYLYVIGEHKLE</td></tr>       <tr><td align="center"><font face="Arial">1</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3tma" target="_blank">3tmaA</a></td><td align ="center"><font face="Arial"> 0.16</font></td><td><font face="Arial"> 0.24</font></td><td><font face="Arial"> 0.92</font></td><td align="center"><font face="Arial"> 1.72</font></td><td align="center"><a href="threading1.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">LRG--SL-----TPVLAQAL<font color="#0F820F">L</font>RL<font color="#333333">A</font>DARPGMR<font color="#0F820F">V</font>L<font color="#E60A0A">D</font>PFT<font color="#666666">G</font>S<font color="#666666">G</font>T<font color="#0F820F">I</font>ALE<font color="#333333">A</font><font color="#333333">A</font>STLGTSPVYAGDLDEK<font color="#145AFF">R</font>LGL<font color="#333333">A</font>REAALAS<font color="#666666">G</font>L-SWIRFLRADARHLFFP<font color="#E60A0A">E</font>VDRILANP<font color="#0F820F">L</font>FHLYWDFLRGALAL<font color="#0F820F">L</font>P<font color="#D69682">P</font><font color="#666666">G</font>G<font color="#145AFF">R</font><font color="#0F820F">V</font>ALLTLLPP<font color="#666666">G</font>FALRHARV<font color="#0F820F">V</font><font color="#E60A0A">E</font>Q<font color="#666666">G</font>GVYPR<font color="#0F820F">V</font>FVLEKL-----</td></tr>

       <tr><td align="center"><font face="Arial">2</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1u2z" target="_blank">1u2zA</a></td><td align ="center"><font face="Arial"> 0.19</font></td><td><font face="Arial"> 0.26</font></td><td><font face="Arial"> 0.95</font></td><td align="center"><font face="Arial"> 1.95</font></td><td align="center"><a href="threading2.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--------<font color="#3232AA">Y</font>GELLPNFLSDVYQQCQLKKG<font color="#E60A0A">D</font>TFM<font color="#E60A0A">D</font><font color="#0F820F">L</font><font color="#666666">G</font>S<font color="#666666">G</font>V<font color="#666666">G</font>NCVVQ<font color="#333333">A</font><font color="#333333">A</font>LECGCALSF<font color="#666666">G</font>C<font color="#E60A0A">E</font><font color="#0F820F">I</font>M<font color="#E60A0A">D</font>DASDLTILQY<font color="#E60A0A">E</font>EMRLNNVEFSL<font color="#145AFF">K</font>KS<font color="#3232AA">F</font>VAEL<font color="#0F820F">I</font>PQCD<font color="#0F820F">V</font>ILV<font color="#3232AA">F</font><font color="#0F820F">L</font>FDEDLNKKVE<font color="#145AFF">K</font>ILQTA<font color="#145AFF">K</font>V<font color="#666666">G</font>CKII<font color="#FA9600">S</font>LKSLRSLTYQIK<font color="#0F820F">V</font>QR<font color="#0F820F">V</font>SWTHSGGEY<font color="#3232AA">Y</font>ISTVM<font color="#E60A0A">E</font>DVD<font color="#E60A0A">E</font></td></tr>

       <tr><td align="center"><font face="Arial">3</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3grz" target="_blank">3grzB</a></td><td align ="center"><font face="Arial"> 0.17</font></td><td><font face="Arial"> 0.21</font></td><td><font face="Arial"> 0.93</font></td><td align="center"><font face="Arial"> 1.66</font></td><td align="center"><a href="threading3.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">IRLD<font color="#D69682">P</font>GLAF--GNHQTTQLAMLGIERVKPLT<font color="#0F820F">V</font>A<font color="#E60A0A">D</font>V<font color="#666666">G</font>T<font color="#666666">G</font>S<font color="#666666">G</font>ILA<font color="#0F820F">I</font>A<font color="#333333">A</font>H<font color="#145AFF">K</font>L-GA<font color="#145AFF">K</font>SVLATD<font color="#0F820F">I</font>S<font color="#E60A0A">D</font><font color="#E60A0A">E</font>SMTA<font color="#333333">A</font>EE<font color="#00DCDC">N</font>AAL<font color="#00DCDC">N</font><font color="#666666">G</font>IYDI<font color="#333333">A</font>LQKTSLLAD<font color="#0F820F">V</font><font color="#E60A0A">D</font>-GKFDLIVANI<font color="#0F820F">L</font>AEILLD<font color="#0F820F">L</font>I<font color="#D69682">P</font>Q<font color="#0F820F">L</font>DSH<font color="#0F820F">L</font>NEDGQ<font color="#0F820F">V</font>IFSGIDYNSFQIDLKMRAGR---WIGLAISRKH-----</td></tr>

       <tr><td align="center"><font face="Arial">4</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3fpf" target="_blank">3fpfA</a></td><td align ="center"><font face="Arial"> 0.13</font></td><td><font face="Arial"> 0.20</font></td><td><font face="Arial"> 0.97</font></td><td align="center"><font face="Arial"> 0.82</font></td><td align="center"><a href="threading4.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">VIDLRSFYFY<font color="#D69682">P</font>RYLELLKNEAALGRFRRGERAVFI<font color="#666666">G</font>G<font color="#666666">G</font>PLPLTGILLSHVYGMRVNV<font color="#0F820F">V</font><font color="#E60A0A">E</font><font color="#0F820F">I</font>EPDIAELSRKV<font color="#0F820F">I</font><font color="#E60A0A">E</font>GL<font color="#666666">G</font><font color="#0F820F">V</font>D-GVNVIT<font color="#666666">G</font>DETVI<font color="#E60A0A">D</font>GL<font color="#E60A0A">E</font>FD<font color="#0F820F">V</font>LMVAA<font color="#0F820F">L</font>AEPKRRVFRNIHRYVDTE<font color="#FA9600">T</font><font color="#145AFF">R</font>IIYRTYTGMRAA<font color="#D69682">P</font>VRRAG<font color="#0F820F">V</font>LPSGKVNNSV<font color="#0F820F">L</font>VFKCP----</td></tr>

       <tr><td align="center"><font face="Arial">5</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2nxc" target="_blank">2nxcA</a></td><td align ="center"><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.28</font></td><td><font face="Arial"> 0.93</font></td><td align="center"><font face="Arial"> 1.15</font></td><td align="center"><a href="threading5.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">LVIE<font color="#D69682">P</font>G-----GHHETT<font color="#145AFF">R</font>LA<font color="#0F820F">L</font>KALALRPG<font color="#E60A0A">D</font>K<font color="#0F820F">V</font>L<font color="#E60A0A">D</font><font color="#0F820F">L</font><font color="#666666">G</font>T<font color="#666666">G</font>S<font color="#666666">G</font>VLA<font color="#0F820F">I</font>A<font color="#333333">A</font>E<font color="#145AFF">K</font>-LGG<font color="#145AFF">K</font>-<font color="#333333">A</font>L<font color="#666666">G</font><font color="#0F820F">V</font>D<font color="#0F820F">I</font>DPMVLPQ<font color="#333333">A</font>E<font color="#333333">A</font><font color="#00DCDC">N</font>AKR<font color="#00DCDC">N</font><font color="#666666">G</font><font color="#0F820F">V</font>--<font color="#145AFF">R</font>PRFLE<font color="#666666">G</font>SLEAALFGPFDLLVAN<font color="#0F820F">L</font>YAELHAA<font color="#0F820F">L</font>A<font color="#D69682">P</font>RYREA<font color="#0F820F">L</font>V<font color="#D69682">P</font><font color="#666666">G</font>G<font color="#145AFF">R</font>ALLTGIL--KDRAPL<font color="#0F820F">V</font>REAMA<font color="#666666">G</font>AGFRPLEEAAE<font color="#666666">G</font><font color="#E60A0A">E</font>WV<font color="#0F820F">L</font>L</td></tr>

       <tr><td align="center"><font face="Arial">6</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3tma" target="_blank">3tmaA</a></td><td align ="center"><font face="Arial"> 0.16</font></td><td><font face="Arial"> 0.24</font></td><td><font face="Arial"> 0.91</font></td><td align="center"><font face="Arial"> 1.54</font></td><td align="center"><a href="threading6.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">---------GSLTPVLAQAL<font color="#0F820F">L</font>RL<font color="#333333">A</font>DARPGMR<font color="#0F820F">V</font>L<font color="#E60A0A">D</font>PFT<font color="#666666">G</font>S<font color="#666666">G</font>T<font color="#0F820F">I</font>ALE<font color="#333333">A</font><font color="#333333">A</font>STLGTSPVYAGDLDEK<font color="#145AFF">R</font>LGL<font color="#333333">A</font>REAALAS<font color="#666666">G</font>L-SWIRFLRADARHLPRFFPE<font color="#0F820F">V</font>DRILA<font color="#0F820F">L</font>FHLYWDFLRGALAL<font color="#0F820F">L</font>P<font color="#D69682">P</font><font color="#666666">G</font>G<font color="#145AFF">R</font><font color="#0F820F">V</font>ALLTLRPP<font color="#666666">G</font>FALRHARV<font color="#0F820F">V</font><font color="#E60A0A">E</font>Q<font color="#666666">G</font>GVYPR<font color="#0F820F">V</font>FVLEKL-----</td></tr>

       <tr><td align="center"><font face="Arial">7</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1u2z" target="_blank">1u2zA</a></td><td align ="center"><font face="Arial"> 0.16</font></td><td><font face="Arial"> 0.26</font></td><td><font face="Arial"> 0.99</font></td><td align="center"><font face="Arial"> 0.89</font></td><td align="center"><a href="threading7.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">KAF-SNYV<font color="#3232AA">Y</font>GELLPNFLSDVYQQCQLKKG<font color="#E60A0A">D</font>TFM<font color="#E60A0A">D</font><font color="#0F820F">L</font><font color="#666666">G</font>S<font color="#666666">G</font>V<font color="#666666">G</font>NCVVQ<font color="#333333">A</font><font color="#333333">A</font>LECGCALSF<font color="#666666">G</font>C<font color="#E60A0A">E</font><font color="#0F820F">I</font>M<font color="#E60A0A">D</font>LQYE<font color="#E60A0A">E</font>LKKRCKLY<font color="#666666">G</font>MRNNVEFL<font color="#145AFF">K</font>KS<font color="#3232AA">F</font>VDNNRPQCD<font color="#0F820F">V</font>ILVN<font color="#0F820F">L</font>FDEDLNKKVE<font color="#145AFF">K</font>ILQTA<font color="#145AFF">K</font>V<font color="#666666">G</font>CKII<font color="#FA9600">S</font>LKSLRSLYQRLK<font color="#0F820F">V</font>QRYSVSWSGGEY<font color="#3232AA">Y</font>ISTVM<font color="#E60A0A">E</font>DVD-</td></tr>

       <tr><td align="center"><font face="Arial">8</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2nxc" target="_blank">2nxcA</a></td><td align ="center"><font face="Arial"> 0.24</font></td><td><font face="Arial"> 0.00</font></td><td><font face="Arial"> 0.90</font></td><td align="center"><font face="Arial"> 1.57</font></td><td align="center"><a href="threading8.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">IE--<font color="#D69682">P</font>G-----GHHETT<font color="#145AFF">R</font>LA<font color="#0F820F">L</font>KALALRPG<font color="#E60A0A">D</font>K<font color="#0F820F">V</font>L<font color="#E60A0A">D</font><font color="#0F820F">L</font><font color="#666666">G</font>T<font color="#666666">G</font>S<font color="#666666">G</font>VLA<font color="#0F820F">I</font>A<font color="#333333">A</font>E<font color="#145AFF">K</font>LGG--<font color="#145AFF">K</font><font color="#333333">A</font>L<font color="#666666">G</font><font color="#0F820F">V</font>D<font color="#0F820F">I</font>DPMVLPQ<font color="#333333">A</font>E<font color="#333333">A</font><font color="#00DCDC">N</font>AKR<font color="#00DCDC">N</font><font color="#666666">G</font><font color="#0F820F">V</font>RP--RFLE<font color="#666666">G</font>SLEALPFGPFDLLVAN<font color="#0F820F">L</font>YAELHAA<font color="#0F820F">L</font>A<font color="#D69682">P</font>RYREA<font color="#0F820F">L</font>V<font color="#D69682">P</font><font color="#666666">G</font>G<font color="#145AFF">R</font>ALLTGILKDRAPLVR--EAMA<font color="#666666">G</font>AGFRPLEEAAE<font color="#666666">G</font><font color="#E60A0A">E</font>---W</td></tr>

       <tr><td align="center"><font face="Arial">9</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2gh1" target="_blank">2gh1A</a></td><td align ="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.21</font></td><td><font face="Arial"> 0.94</font></td><td align="center"><font face="Arial"> 1.37</font></td><td align="center"><a href="threading9.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--------<font color="#3232AA">Y</font>YYNDDY<font color="#0F820F">V</font>SFLVNTVW<font color="#145AFF">K</font>ITKPVHIV<font color="#E60A0A">D</font>Y<font color="#666666">G</font><font color="#FFCC00">C</font><font color="#666666">G</font>Y<font color="#666666">G</font>YLGLVLPLLPEGS<font color="#145AFF">K</font>YT<font color="#666666">G</font>IDSGETLLA<font color="#E60A0A">E</font><font color="#333333">A</font>RELFRLLPYD--SEFLE<font color="#666666">G</font>DAT<font color="#E60A0A">E</font>IELNDYDIAICHAFLLHTTPETLQKIHSV<font color="#145AFF">K</font>K<font color="#666666">G</font>GKIICF<font color="#E60A0A">E</font>PASYDG<font color="#00DCDC">N</font>IGKIPIYLSELGVKNIECR<font color="#0F820F">V</font>SDKVNYA</td></tr>

       <tr><td align="center"><font face="Arial">10</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1u2z" target="_blank">1u2zA</a></td><td align ="center"><font face="Arial"> 0.15</font></td><td><font face="Arial"> 0.26</font></td><td><font face="Arial"> 1.00</font></td><td align="center"><font face="Arial"> 2.02</font></td><td align="center"><a href="threading10.pdb"><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">QH<font color="#3232AA">Y</font>KAFYV<font color="#3232AA">Y</font>GELLPNFLSDVYQQCQLKKG<font color="#E60A0A">D</font>TFM<font color="#E60A0A">D</font><font color="#0F820F">L</font><font color="#666666">G</font>S<font color="#666666">G</font>V<font color="#666666">G</font>NCVVQ<font color="#333333">A</font><font color="#333333">A</font>LECGCALSF<font color="#666666">G</font>C<font color="#E60A0A">E</font><font color="#0F820F">I</font>M<font color="#E60A0A">D</font>DASDLTILQY<font color="#E60A0A">E</font>ELRLNNVEFSL<font color="#145AFF">K</font>KSRVAEL<font color="#0F820F">I</font>PQCD<font color="#0F820F">V</font>ILVNNFDEDLNKKVE<font color="#145AFF">K</font>ILQTA<font color="#145AFF">K</font>V<font color="#666666">G</font>CKII<font color="#FA9600">S</font>LKSLRSLTYQIK<font color="#0F820F">V</font>QRYDLKHSGGEY<font color="#3232AA">Y</font>ISTVM<font color="#E60A0A">E</font>SPAR</td></tr>

      <tr><td colspan="9" align="left">

        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">

        <tr><td valign="top">(a)</td><td align="justify">All the residues are colored in black; however, those residues in template which are identical to the residue in the query sequence are highlighted in color. Coloring scheme is based on the property of amino acids, where polar are brightly coloured while non-polar residues are colored in dark shade. (<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/color.html" target="_blank">more about the colors used</a>)</td></tr>

        <tr><td valign="top">(b)</td><td align="justify">Rank of templates represents the top ten threading templates used by I-TASSER.</td></tr>

        <tr><td valign="top">(c)</td><td align="justify">Ident1 is the percentage sequence identity of the templates in the threading aligned region with the query sequence.</td></tr>

        <tr><td valign="top">(d)</td><td align="justify">Ident2 is the percentage sequence identity of the whole template chains with  query sequence.</td></tr>

        <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the threading alignment and is equal to the number of aligned residues divided by the length of query protein.</td></tr>

        <tr><td valign="top">(f)</td><td align="justify">Norm. Z-score is the normalized Z-score of the threading alignments. Alignment with a Normalized Z-score >1 mean a good alignment and vice versa.</td></tr>

        <tr><td valign="top">(g)</td><td align="justify">Download Align. provides the 3D structure of the aligned regions of the threading templates.</td></tr>

        <tr><td valign="top">(h)</td><td align="justify">The top 10 alignments reported above (in order of their ranking) are from the following threading programs:</td></tr>

        <tr><td>&nbsp;</td><td align ="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

1: MUSTER&nbsp;&nbsp;&nbsp;2: FFAS-3D&nbsp;&nbsp;&nbsp;3: SPARKS-X&nbsp;&nbsp;&nbsp;4: HHSEARCH2&nbsp;&nbsp;&nbsp;5: HHSEARCH I&nbsp;&nbsp;&nbsp;6: Neff-PPAS&nbsp;&nbsp;&nbsp;7: HHSEARCH&nbsp;&nbsp;&nbsp;8: pGenTHREADER&nbsp;&nbsp;&nbsp;9: wdPPAS&nbsp;&nbsp;&nbsp;10: PROSPECT2&nbsp;&nbsp;&nbsp;        </td></tr>

      </table>

      </td></tr></table>

   </div>

</td></tr>

  <tr><td>

   <br/>

   <div style="background:#6599FF;width:350px;">

   <table><tr><td style= "valign:middle;">

   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Top 5 Models predicted by I-TASSER</font>

   </td></tr></table>

   </div>

   <br/>

   <div style="position:relative;left:30px;">

       <table  style="table-layout:fixed;width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">

  <tr> <td align=center><b>Generated 3D models</b></td> <td colspan=3 align=center> <a href=lscore.txt target="_blank"> Estimated local accuracy of models</a> </td></tr>

       <tr>       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(300, "load model1.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");</script></td>

<td  colspan=3 align=center >  <img src=RSQ_1.png>  </td>   </tr>

 <tr> <td  align =left>

<li><a href="model1.pdb">Download Model 1</a>



<li>C-score=-0.57 (<a href="cscore.txt" target="_blank" >Read more about C-score</a>) 



<li>Estimated TM-score = 0.64&plusmn;0.13



<li>Estimated RMSD = 6.1&plusmn;3.8&Aring;



<br><br>



</td>



 

</tr>

       <tr>       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(300, "load model2.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");</script></td>

<td  colspan=3 align=center >  <img src=RSQ_2.png>  </td>   </tr>

 <tr> <td  align =left>

<li><a href="model2.pdb">Download Model 2</a>



<li>C-score = -1.58 <br><br>



</td> </tr>

       <tr>       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(300, "load model3.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");</script></td>

<td  colspan=3 align=center >  <img src=RSQ_3.png>  </td>   </tr>

 <tr> <td  align =left>

<li><a href="model3.pdb">Download Model 3</a>



<li>C-score = -2.80 <br><br>



</td> </tr>

       <tr>       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(300, "load model4.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");</script></td>

<td  colspan=3 align=center >  <img src=RSQ_4.png>  </td>   </tr>

 <tr> <td  align =left>

<li><a href="model4.pdb">Download Model 4</a>



<li>C-score = -1.20 <br><br>



</td> </tr>

       <tr>       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(300, "load model5.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");</script></td>

<td  colspan=3 align=center >  <img src=RSQ_5.png>  </td>   </tr>

 <tr> <td  align =left>

<li><a href="model5.pdb">Download Model 5</a>



<li>C-score = -2.12 <br><br>



</td> </tr>

       </table>

    </div>

    </div>

  </td></tr>

  <tr><td>

   <br/>

   <div style="background:#6599FF;width:650px;">

      <table>

         <tr><td style= "valign:middle;"><font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Proteins with highly similar structure in PDB (as identified by <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">TM-align</a>)</font></td></tr>

      </table>

   </div>

   <br/>

   <div>

   <div style="position:relative;left:30px;background:#F2F2F2;">

       <table border="0" style="background:#F2F2F2;align="left">

           <tr>

           <td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model1_1o54A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;");</script><br/><SCRIPT>jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>

           <td valign="top"><b>Top 10 Identified stuctural analogs in PDB</b><br/><br/>

                 <table border="0" style="font-family:Arial;font-size:13px;">

                    <tr><th>Click <br>to view</th><th >Rank</th><th >PDB<br/>Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th>Cov.</th><th >Download<br/>Alignment</th></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1o54A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", true);</script></td><font face="Monospace">

                       <td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1o54" target="_blank">1o54A</a></font></td><td align="center">0.817</td><td valign="middle" align="center"> 2.32</td><td valign="middle" align="center">0.153</td><td valign="middle">0.957</td><td><a href="model1_1o54A.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3tmaA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3tma" target="_blank">3tmaA</a></font></td><td align="center">0.815</td><td valign="middle" align="center"> 2.23</td><td valign="middle" align="center">0.132</td><td valign="middle">0.932</td><td><a href="model1_3tmaA.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1i9gA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1i9g" target="_blank">1i9gA</a></font></td><td align="center">0.811</td><td valign="middle" align="center"> 2.54</td><td valign="middle" align="center">0.095</td><td valign="middle">0.969</td><td><a href="model1_1i9gA.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3axsA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3axs" target="_blank">3axsA</a></font></td><td align="center">0.810</td><td valign="middle" align="center"> 2.62</td><td valign="middle" align="center">0.125</td><td valign="middle">0.982</td><td><a href="model1_3axsA.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3mb5A2.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3mb5" target="_blank">3mb5A2</a></font></td><td align="center">0.806</td><td valign="middle" align="center"> 2.30</td><td valign="middle" align="center">0.195</td><td valign="middle">0.945</td><td><a href="model1_3mb5A2.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2ytzA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2ytz" target="_blank">2ytzA</a></font></td><td align="center">0.802</td><td valign="middle" align="center"> 2.66</td><td valign="middle" align="center">0.075</td><td valign="middle">0.976</td><td><a href="model1_2ytzA.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2f8lA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2f8l" target="_blank">2f8lA</a></font></td><td align="center">0.798</td><td valign="middle" align="center"> 2.81</td><td valign="middle" align="center">0.138</td><td valign="middle">0.976</td><td><a href="model1_2f8lA.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1u2zA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1u2z" target="_blank">1u2zA</a></font></td><td align="center">0.797</td><td valign="middle" align="center"> 2.94</td><td valign="middle" align="center">0.148</td><td valign="middle">0.994</td><td><a href="model1_1u2zA.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2pwyB.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2pwy" target="_blank">2pwyB</a></font></td><td align="center">0.795</td><td valign="middle" align="center"> 2.56</td><td valign="middle" align="center">0.141</td><td valign="middle">0.957</td><td><a href="model1_2pwyB.pdb"><font face="Arial">Download</font></a></td></tr>

               

                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3lgaA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">

                       <td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3lga" target="_blank">3lgaA</a></font></td><td align="center">0.795</td><td valign="middle" align="center"> 2.30</td><td valign="middle" align="center">0.197</td><td valign="middle">0.932</td><td><a href="model1_3lgaA.pdb"><font face="Arial">Download</font></a></td></tr>

               </table><br/>

               <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">

                <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>

                <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>

                <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>

                <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>

                <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>

               </table>

          </td></tr>

       </table>

    </div>

    </div>

  </td></tr>

  <tr><td>

  <br/><br/>

   <div style="background:#6599FF;width:400px;">

   <table><tr><td style= "valign:middle;">

      <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Function Prediction using <a href="http://zhanglab.ccmb.med.umich.edu/COFACTOR" target="_blank">COFACTOR</a></font>

   </td></tr></table>

   </div>

  <br/>

   <div style="position:relative;left:30px;background:#FF9900; width:320px;">

   <table><tr><td style= "valign:middle;">

        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted EC Numbers</font>

   </td></tr></table>

   </div>

   <br/>

  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">

       <table border="0" style="background:#F2F2F2;align="left">

       <tr><td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(450, "load model1.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons; select 36; wireframe 70; spacefill 100; color cpk;  select 36 and *.ca; label %m%r; color label magenta; select 59; wireframe 70; spacefill 100; color cpk;  select 59 and *.ca; label %m%r; color label magenta;      set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT>jmolCheckbox("spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>

           <td valign="top"><b>Top 5 enzyme homologs in PDB</b><br/><br/>

             <table  border="0" style="font-family:Arial;font-size:13px;"><tr><th >Click <br>to view</th><th >Rank</th><th >Cscore<sup>EC</sup></th><th>PDB<br/>Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th >Cov.</th><th >EC Number</th><th>Predicted Active Site Residues</th></tr>

         <tr>

         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 36; wireframe 70; spacefill 100; color cpk;  select 36 and *.ca; label %m%r; color label magenta; select 59; wireframe 70; spacefill 100; color cpk;  select 59 and *.ca; label %m%r; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>

         <td align="left"><font face="Arial">1</font></td><td align="center"><font face="Arial">0.413</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1zq9" target="_blank">1zq9A</a></td><td align="center"><font face="Arial">0.789</font></td><td align="center"><font face="Arial"> 2.51</font></td><td align="center"><font face="Arial">0.137</font></td><td align="right"><font face="Arial">0.939</font></td>

         <td align="left"><span title="NA"><a href="http://ca.expasy.org/enzyme/2.1.1.-" target="_blank"><font face="Arial">2.1.1.-</font></a></span>&nbsp;         </td>

         <td align="left"><font face="Arial">36,59</font></td>         </tr>

         <tr>

         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 7; wireframe 70; spacefill 100; color cpk;  select 7 and *.ca; label %m%r; color label magenta; select 36; wireframe 70; spacefill 100; color cpk;  select 36 and *.ca; label %m%r; color label magenta; select 38; wireframe 70; spacefill 100; color cpk;  select 38 and *.ca; label %m%r; color label magenta; select 62; wireframe 70; spacefill 100; color cpk;  select 62 and *.ca; label %m%r; color label magenta; select 86; wireframe 70; spacefill 100; color cpk;  select 86 and *.ca; label %m%r; color label magenta; select 88; wireframe 70; spacefill 100; color cpk;  select 88 and *.ca; label %m%r; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>

         <td align="left"><font face="Arial">2</font></td><td align="center"><font face="Arial">0.376</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1nw3" target="_blank">1nw3A</a></td><td align="center"><font face="Arial">0.778</font></td><td align="center"><font face="Arial"> 3.10</font></td><td align="center"><font face="Arial">0.205</font></td><td align="right"><font face="Arial">0.988</font></td>

         <td align="left"><span title="Histone-lysine N-methyltransferase."><a href="http://ca.expasy.org/enzyme/2.1.1.43" target="_blank"><font face="Arial">2.1.1.43</font></a></span>&nbsp;         </td>

         <td align="left"><font face="Arial">7,36,38,62,86,88</font></td>         </tr>

         <tr>

         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 36; wireframe 70; spacefill 100; color cpk;  select 36 and *.ca; label %m%r; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>

         <td align="left"><font face="Arial">3</font></td><td align="center"><font face="Arial">0.345</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2nxe" target="_blank">2nxeA</a></td><td align="center"><font face="Arial">0.748</font></td><td align="center"><font face="Arial"> 2.73</font></td><td align="center"><font face="Arial">0.213</font></td><td align="right"><font face="Arial">0.908</font></td>

         <td align="left"><span title="NA"><a href="http://ca.expasy.org/enzyme/2.1.1.-" target="_blank"><font face="Arial">2.1.1.-</font></a></span>&nbsp;         </td>

         <td align="left"><font face="Arial">36</font></td>         </tr>

         <tr>

         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 13; wireframe 70; spacefill 100; color cpk;  select 13 and *.ca; label %m%r; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>

         <td align="left"><font face="Arial">4</font></td><td align="center"><font face="Arial">0.325</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2ytz" target="_blank">2ytzA</a></td><td align="center"><font face="Arial">0.802</font></td><td align="center"><font face="Arial"> 2.66</font></td><td align="center"><font face="Arial">0.075</font></td><td align="right"><font face="Arial">0.976</font></td>

         <td align="left"><span title="tRNA (guanine-N(2)-)-methyltransferase."><a href="http://ca.expasy.org/enzyme/2.1.1.32" target="_blank"><font face="Arial">2.1.1.32</font></a></span>&nbsp;         </td>

         <td align="left"><font face="Arial">13</font></td>         </tr>

         <tr>

         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 32; wireframe 70; spacefill 100; color cpk;  select 32 and *.ca; label %m%r; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>

         <td align="left"><font face="Arial">5</font></td><td align="center"><font face="Arial">0.305</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1o54" target="_blank">1o54A</a></td><td align="center"><font face="Arial">0.817</font></td><td align="center"><font face="Arial"> 2.32</font></td><td align="center"><font face="Arial">0.153</font></td><td align="right"><font face="Arial">0.957</font></td>

         <td align="left"><span title="NA"><a href="http://ca.expasy.org/enzyme/2.1.1.-" target="_blank"><font face="Arial">2.1.1.-</font></a></span>&nbsp;         </td>

         <td align="left"><font face="Arial">32</font></td>         </tr>

         </table><br/>

         <table border="0"  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">

          <tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>

           <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the Enzyme Classification (EC) number prediction. Cscore<sup>EC</sup> values range in between [0-1]; where a higher score indicates a more reliable EC number prediction.</td></tr>

           <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>

           <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>

           <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>

           <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>

         </table>

        </td></tr></table>

  </div>

   </td></tr></table>

</td></tr>

  <tr><td>

  <br/>

   <div style="position:relative;left:30px;background:#FF9900; width:320px;">

      <table><tr><td  valign="middle;">

        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted GO terms</font>

      </td></tr></table>

   </div>

  <br/>

  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">

   <table border="0" style="font-family:Monospace;font-size:13px;background:#F2F2F2">

  <tr><th align="left" valign="top"><font face="Arial">Rank</font></th><th valign="top" align="center"><font face="Arial">Cscore<sup>GO</sup></font></th><th  valign="top" align="center"><font face="Arial">TMscore</font></th><th  valign="top" align="center"><font face="Arial">RMSD<sup>a</sup></font></th><th  valign="top" align="center"><font face="Arial">IDEN<sup>a</sup></font></th><th valign="top" align="center"><font face="Arial">Cov.</font></th><th valign="top" align="center"><font face="Arial">PDB<br/>Hit</font></th><th align="left" valign="top"><font face="Arial">Associated GO Terms</font></th></tr>

			<tr><td align="center" valign="top"><font face="Arial">1</font></td><td align="center"  valign="top"><font face="Arial"> 0.41</font></td><td  valign="top" align="center"><font face="Arial">0.7719</font></td><td align="center"  valign="top"><font face="Arial"> 2.66</font></td><td align="center"  valign="top"><font face="Arial"> 0.17</font></td><td  valign="top" align="center"><font face="Arial"> 0.92</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3grz" target="_blank">3grzB</a></td><td valign="top"><font face="Arial">

<span title="cytoplasm"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0005737 target="_blank">GO:0005737</a></span>

<span title="protein methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0006479 target="_blank">GO:0006479</a></span>

<span title="protein methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008276 target="_blank">GO:0008276</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">2</font></td><td align="center"  valign="top"><font face="Arial"> 0.41</font></td><td  valign="top" align="center"><font face="Arial">0.8056</font></td><td align="center"  valign="top"><font face="Arial"> 2.30</font></td><td align="center"  valign="top"><font face="Arial"> 0.20</font></td><td  valign="top" align="center"><font face="Arial"> 0.94</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3mb5" target="_blank">3mb5A</a></td><td valign="top"><font face="Arial">

<span title="transferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016740 target="_blank">GO:0016740</a></span>

<span title="methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0032259 target="_blank">GO:0032259</a></span>

<span title="tRNA processing"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008033 target="_blank">GO:0008033</a></span>

<span title="tRNA methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0030488 target="_blank">GO:0030488</a></span>

<span title="methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008168 target="_blank">GO:0008168</a></span>

<span title="tRNA (adenine-N1-)-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016429 target="_blank">GO:0016429</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">3</font></td><td align="center"  valign="top"><font face="Arial"> 0.39</font></td><td  valign="top" align="center"><font face="Arial">0.7894</font></td><td align="center"  valign="top"><font face="Arial"> 2.72</font></td><td align="center"  valign="top"><font face="Arial"> 0.14</font></td><td  valign="top" align="center"><font face="Arial"> 0.96</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2pwy" target="_blank">2pwyA</a></td><td valign="top"><font face="Arial">

<span title="transferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016740 target="_blank">GO:0016740</a></span>

<span title="cellular_component"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0005575 target="_blank">GO:0005575</a></span>

<span title="tRNA methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0030488 target="_blank">GO:0030488</a></span>

<span title="tRNA processing"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008033 target="_blank">GO:0008033</a></span>

<span title="methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0032259 target="_blank">GO:0032259</a></span>

<span title="tRNA (adenine-N1-)-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016429 target="_blank">GO:0016429</a></span>

<span title="methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008168 target="_blank">GO:0008168</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">4</font></td><td align="center"  valign="top"><font face="Arial"> 0.38</font></td><td  valign="top" align="center"><font face="Arial">0.7783</font></td><td align="center"  valign="top"><font face="Arial"> 3.10</font></td><td align="center"  valign="top"><font face="Arial"> 0.20</font></td><td  valign="top" align="center"><font face="Arial"> 0.99</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1nw3" target="_blank">1nw3A</a></td><td valign="top"><font face="Arial">

<span title="histone-lysine N-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0018024 target="_blank">GO:0018024</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">5</font></td><td align="center"  valign="top"><font face="Arial"> 0.38</font></td><td  valign="top" align="center"><font face="Arial">0.8106</font></td><td align="center"  valign="top"><font face="Arial"> 2.54</font></td><td align="center"  valign="top"><font face="Arial"> 0.10</font></td><td  valign="top" align="center"><font face="Arial"> 0.97</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1i9g" target="_blank">1i9gA</a></td><td valign="top"><font face="Arial">

<span title="S-adenosylmethionine-dependent methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008757 target="_blank">GO:0008757</a></span>

<span title="tRNA (adenine-N1-)-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016429 target="_blank">GO:0016429</a></span>

<span title="protein homotetramerization"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0051289 target="_blank">GO:0051289</a></span>

<span title="methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008168 target="_blank">GO:0008168</a></span>

<span title="tRNA methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0030488 target="_blank">GO:0030488</a></span>

<span title="transferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016740 target="_blank">GO:0016740</a></span>

<span title="plasma membrane"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0005886 target="_blank">GO:0005886</a></span>

<span title="tRNA processing"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008033 target="_blank">GO:0008033</a></span>

<span title="methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0032259 target="_blank">GO:0032259</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">6</font></td><td align="center"  valign="top"><font face="Arial"> 0.37</font></td><td  valign="top" align="center"><font face="Arial">0.7470</font></td><td align="center"  valign="top"><font face="Arial"> 2.60</font></td><td align="center"  valign="top"><font face="Arial"> 0.18</font></td><td  valign="top" align="center"><font face="Arial"> 0.90</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3hm2" target="_blank">3hm2A</a></td><td valign="top"><font face="Arial">

<span title="protein methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008276 target="_blank">GO:0008276</a></span>

<span title="cobalamin biosynthetic process"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0009236 target="_blank">GO:0009236</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">7</font></td><td align="center"  valign="top"><font face="Arial"> 0.36</font></td><td  valign="top" align="center"><font face="Arial">0.7955</font></td><td align="center"  valign="top"><font face="Arial"> 2.90</font></td><td align="center"  valign="top"><font face="Arial"> 0.15</font></td><td  valign="top" align="center"><font face="Arial"> 0.99</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1u2z" target="_blank">1u2zC</a></td><td valign="top"><font face="Arial">

<span title="histone-lysine N-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0018024 target="_blank">GO:0018024</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">8</font></td><td align="center"  valign="top"><font face="Arial"> 0.35</font></td><td  valign="top" align="center"><font face="Arial">0.8169</font></td><td align="center"  valign="top"><font face="Arial"> 2.32</font></td><td align="center"  valign="top"><font face="Arial"> 0.15</font></td><td  valign="top" align="center"><font face="Arial"> 0.96</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1o54" target="_blank">1o54A</a></td><td valign="top"><font face="Arial">

<span title="tRNA (adenine-N1-)-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016429 target="_blank">GO:0016429</a></span>

<span title="tRNA methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0030488 target="_blank">GO:0030488</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">9</font></td><td align="center"  valign="top"><font face="Arial"> 0.35</font></td><td  valign="top" align="center"><font face="Arial">0.7869</font></td><td align="center"  valign="top"><font face="Arial"> 2.63</font></td><td align="center"  valign="top"><font face="Arial"> 0.12</font></td><td  valign="top" align="center"><font face="Arial"> 0.96</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3bwm" target="_blank">3bwmA</a></td><td valign="top"><font face="Arial">

<span title="magnesium ion binding"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0000287 target="_blank">GO:0000287</a></span>

<span title="catecholamine metabolic process"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0006584 target="_blank">GO:0006584</a></span>

<span title="O-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008171 target="_blank">GO:0008171</a></span>

<span title="catechol O-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016206 target="_blank">GO:0016206</a></span>

<span title="neurotransmitter catabolic process"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0042135 target="_blank">GO:0042135</a></span>

</font></td></tr>

			<tr><td align="center" valign="top"><font face="Arial">10</font></td><td align="center"  valign="top"><font face="Arial"> 0.35</font></td><td  valign="top" align="center"><font face="Arial">0.7891</font></td><td align="center"  valign="top"><font face="Arial"> 2.51</font></td><td align="center"  valign="top"><font face="Arial"> 0.14</font></td><td  valign="top" align="center"><font face="Arial"> 0.94</font></td><td align="center"  valign="top"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1zq9" target="_blank">1zq9A</a></td><td valign="top"><font face="Arial">

<span title="methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008168 target="_blank">GO:0008168</a></span>

<span title="RNA binding"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0003723 target="_blank">GO:0003723</a></span>

<span title="transferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016740 target="_blank">GO:0016740</a></span>

<span title="rRNA processing"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0006364 target="_blank">GO:0006364</a></span>

<span title="rRNA methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0031167 target="_blank">GO:0031167</a></span>

<span title="nucleolus"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0005730 target="_blank">GO:0005730</a></span>

<span title="nucleus"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0005634 target="_blank">GO:0005634</a></span>

<span title="methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0032259 target="_blank">GO:0032259</a></span>

<span title="rRNA modification"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0000154 target="_blank">GO:0000154</a></span>

<span title="rRNA (adenine-N6,N6-)-dimethyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0000179 target="_blank">GO:0000179</a></span>

<span title="rRNA methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008649 target="_blank">GO:0008649</a></span>

<span title="rRNA (adenine) methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016433 target="_blank">GO:0016433</a></span>

</font></td></tr>

   </table>

      <p><table border="0" style="font-family:Monospace;font-size:13px;background:#F2F2F2">

      <tr><th colspan="6" align="left"><font face="Arial">Consensus Prediction of Gene Ontology terms</font><th><td>&nbsp;</td></tr>

      <tr><td>

      <table cellpadding="2" cellspacing="3" style="font-family:Monospace;font-size:13px;background:#F2F2F2">

         <tr><th valign="top" colspan="2"><font face="Arial">Molecular Function</font></th><td>&nbsp;&nbsp;</td><th align"center" valign="top" colspan="2"><font face="Arial">Biological Process</font></th><td>&nbsp;</td><th align"center" colspan="2"><font face="Arial" >Cellular Location</font></tr>

         <tr><th valign="top"><font face="Arial">GO term</font></th><th align"center" valign="top"><font face="Arial">GO-Score</font><td>&nbsp;&nbsp;</td><th valign="top"><font face="Arial">GO term</font></th><th align"center" valign="top"><font face="Arial">GO-Score</font><td>&nbsp;</td><th valign="top"><font face="Arial">GO term</font></th><th align"center" valign="top"><font face="Arial">GO-Score</font></th></tr>

     <tr>

      <td align="center"><font face="Arial"><span title="tRNA (adenine-N1-)-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016429 target="_blank">GO:0016429</a></span></font><td align="center"><font face="Arial">0.78</font></td><td>&nbsp;&nbsp;</td>

      <td align="center"><font face="Arial"><span title="tRNA methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0030488 target="_blank">GO:0030488</font></td><td align="center"></span><font face="Arial">0.78</font></td><td>&nbsp;&nbsp;</td>

      <td align="center"><font face="Arial"><span title="cytoplasm"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0005737 target="_blank">GO:0005737</font></td><td align="center"></span><font face="Arial">0.41</font></td><td>&nbsp;&nbsp;</td>

     </tr>

     <tr>

      <td align="center"><font face="Arial"><span title="histone-lysine N-methyltransferase activity"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0018024 target="_blank">GO:0018024</a></span></font><td align="center"><font face="Arial">0.38</font></td><td>&nbsp;&nbsp;</td>

      <td align="center"><font face="Arial"><span title="protein methylation"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0006479 target="_blank">GO:0006479</font></td><td align="center"></span><font face="Arial">0.41</font></td><td>&nbsp;&nbsp;</td>

      <td align="center"><font face="Arial"><span title="plasma membrane"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0005886 target="_blank">GO:0005886</font></td><td align="center"></span><font face="Arial">0.38</font></td><td>&nbsp;&nbsp;</td>

     </tr>

     <tr>

      <td align="center">&nbsp;&nbsp;</td><td >&nbsp;&nbsp;</td><td>&nbsp;&nbsp;</td>

      <td align="center"><font face="Arial"><span title="protein homotetramerization"><a href=http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0051289 target="_blank">GO:0051289</font></td><td align="center"></span><font face="Arial">0.38</font></td><td>&nbsp;&nbsp;</td>

      <td align="center">&nbsp;&nbsp;</td><td >&nbsp;&nbsp;</td><td>&nbsp;&nbsp;</td>

     </tr>

   </table>

  </td></tr></table>

  <table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">

      <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup>, which is a combined measure for evaluating global and local similarity between query and template protein. Cscore<sup>GO</sup> values range in between [0-1]; where a higher value indicates a better confidence in predicting the function using the template.</td></tr>

      <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>

      <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>

      <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>

      <tr><td valign="top">(e)</td><td align="justify">Cov. represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>

      <tr><td valign="top">(f)</td><td align="justify">The second table shows a consistence of function (GO terms) amongst top scoring templates. The GO-Score associated with each prediction is defined as the average weight of the GO term, where the weights are assigned based on Cscore<sup>GO</sup> of the template from which the GO term is derived.</td></tr>

     </table>

  </div>

</td></tr>

  <tr><td>

  <br/>

   <div style="position:relative;left:30px;background:#FF9900; width:320px;">

      <table><tr><td  valign="middle;">

        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Binding Site</font>

      </td></tr></table>

   </div>

  <br/>

  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">

   <table border="0" style="font-family:Arial;font-size:13px;background:#F2F2F2">

       <tr><td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(450, "load model1_bs1.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons; select hetero; spacefill 95%; set radius 1.0; color cpk;  set ambient 40; select 36,37,38,42,59,60,86,87,102,103; wireframe 75; spacefill 100;select 36 and *.ca; label %m%r; color label magenta;select 37 and *.ca; label %m%r; color label magenta;select 38 and *.ca; label %m%r; color label magenta;select 42 and *.ca; label %m%r; color label magenta;select 59 and *.ca; label %m%r; color label magenta;select 60 and *.ca; label %m%r; color label magenta;select 86 and *.ca; label %m%r; color label magenta;select 87 and *.ca; label %m%r; color label magenta;select 102 and *.ca; label %m%r; color label magenta;select 103 and *.ca; label %m%r; color label magenta;     set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT>jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT>

</td>

       <td valign="top"><b>Template proteins with similar binding site:<b><br/><br/>

       <table border="0" style="font-family:Arial;font-size:13px;"><tr><th>Click <br>to view</th><th >Rank</th><th >Cscore<sup>LB</sup></th><th >PDB<br/>Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th >Cov.</th><th >BS-score</th><th>Lig. Name</th><th >Download<br/>Complex</th><th align="left"> Predicted binding site residues</th></tr>

       <tr>

       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_bs1.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons; select hetero; spacefill 95%; set radius 0.90; color cpk; set ambient 40; select 36,37,38,42,59,60,86,87,102,103; wireframe 75; spacefill 100;select 36:A and *.ca; label %m%r; color label magenta;select 37:A and *.ca; label %m%r; color label magenta;select 38:A and *.ca; label %m%r; color label magenta;select 42:A and *.ca; label %m%r; color label magenta;select 59:A and *.ca; label %m%r; color label magenta;select 60:A and *.ca; label %m%r; color label magenta;select 86:A and *.ca; label %m%r; color label magenta;select 87:A and *.ca; label %m%r; color label magenta;select 102:A and *.ca; label %m%r; color label magenta;select 103:A and *.ca; label %m%r; color label magenta;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>

    <font face="Monospace"><td valign="middle" align="center">1</td><td valign="middle" align="center">0.61</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2zbr" target="_blank">2zbrA</a></font></td><td align="center">0.733</td><td valign="middle" align="center"> 2.57</td><td valign="middle" align="center">0.231</td><td valign="middle">0.877</td><td valign="middle" align="center">1.11</td><td>SFG</td><td><a href="model1_bs1.pdb"><font face="Arial">Download</font></a></td><td valign="middle">36,37,38,42,59,60,86,87,102,103</td></tr>

       <tr>

       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_bs2.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons; select hetero; spacefill 95%; set radius 0.90; color cpk; set ambient 40; select 9,10,11,12,34,36,37,41,42,59,60,86,87,88,89,100,105; wireframe 75; spacefill 100;select 9:A and *.ca; label %m%r; color label magenta;select 10:A and *.ca; label %m%r; color label magenta;select 11:A and *.ca; label %m%r; color label magenta;select 12:A and *.ca; label %m%r; color label magenta;select 34:A and *.ca; label %m%r; color label magenta;select 36:A and *.ca; label %m%r; color label magenta;select 37:A and *.ca; label %m%r; color label magenta;select 41:A and *.ca; label %m%r; color label magenta;select 42:A and *.ca; label %m%r; color label magenta;select 59:A and *.ca; label %m%r; color label magenta;select 60:A and *.ca; label %m%r; color label magenta;select 86:A and *.ca; label %m%r; color label magenta;select 87:A and *.ca; label %m%r; color label magenta;select 88:A and *.ca; label %m%r; color label magenta;select 89:A and *.ca; label %m%r; color label magenta;select 100:A and *.ca; label %m%r; color label magenta;select 105:A and *.ca; label %m%r; color label magenta;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>

    <font face="Monospace"><td valign="middle" align="center">2</td><td valign="middle" align="center">0.35</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3sr4" target="_blank">3sr4A</a></font></td><td align="center">0.777</td><td valign="middle" align="center"> 3.23</td><td valign="middle" align="center">0.209</td><td valign="middle">1.000</td><td valign="middle" align="center">1.08</td><td>TT8</td><td><a href="model1_bs2.pdb"><font face="Arial">Download</font></a></td><td valign="middle">9,10,11,12,34,36,37,41,42,59,60,86,87,88,89,100,105</td></tr>

       <tr>

       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_bs3.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons; select hetero; spacefill 95%; set radius 0.90; color cpk; set ambient 40; select 38,59,61,64,101,108,128,129; wireframe 75; spacefill 100;select 38:A and *.ca; label %m%r; color label magenta;select 59:A and *.ca; label %m%r; color label magenta;select 61:A and *.ca; label %m%r; color label magenta;select 64:A and *.ca; label %m%r; color label magenta;select 101:A and *.ca; label %m%r; color label magenta;select 108:A and *.ca; label %m%r; color label magenta;select 128:A and *.ca; label %m%r; color label magenta;select 129:A and *.ca; label %m%r; color label magenta;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>

    <font face="Monospace"><td valign="middle" align="center">3</td><td valign="middle" align="center">0.23</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3ozr" target="_blank">3ozrA</a></font></td><td align="center">0.784</td><td valign="middle" align="center"> 2.71</td><td valign="middle" align="center">0.121</td><td valign="middle">0.963</td><td valign="middle" align="center">0.83</td><td>OZR</td><td><a href="model1_bs3.pdb"><font face="Arial">Download</font></a></td><td valign="middle">38,59,61,64,101,108,128,129</td></tr>

       <tr>

       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_bs4.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons; select hetero; spacefill 95%; set radius 0.90; color cpk; set ambient 40; select 18,36,38,41,42,58,59,60,61,86,87,88,102,111; wireframe 75; spacefill 100;select 18:A and *.ca; label %m%r; color label magenta;select 36:A and *.ca; label %m%r; color label magenta;select 38:A and *.ca; label %m%r; color label magenta;select 41:A and *.ca; label %m%r; color label magenta;select 42:A and *.ca; label %m%r; color label magenta;select 58:A and *.ca; label %m%r; color label magenta;select 59:A and *.ca; label %m%r; color label magenta;select 60:A and *.ca; label %m%r; color label magenta;select 61:A and *.ca; label %m%r; color label magenta;select 86:A and *.ca; label %m%r; color label magenta;select 87:A and *.ca; label %m%r; color label magenta;select 88:A and *.ca; label %m%r; color label magenta;select 102:A and *.ca; label %m%r; color label magenta;select 111:A and *.ca; label %m%r; color label magenta;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>

    <font face="Monospace"><td valign="middle" align="center">4</td><td valign="middle" align="center">0.19</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2ejt" target="_blank">2ejtA</a></font></td><td align="center">0.795</td><td valign="middle" align="center"> 2.82</td><td valign="middle" align="center">0.075</td><td valign="middle">0.976</td><td valign="middle" align="center">0.89</td><td>SAM</td><td><a href="model1_bs4.pdb"><font face="Arial">Download</font></a></td><td valign="middle">18,36,38,41,42,58,59,60,61,86,87,88,102,111</td></tr>

       <tr>

       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_bs5.pdb; set frank off; select all; color group; spacefill off; wireframe off; cartoons; select hetero; spacefill 95%; set radius 0.90; color cpk; set ambient 40; select 20,100,101,102,128; wireframe 75; spacefill 100;select 20:A and *.ca; label %m%r; color label magenta;select 100:A and *.ca; label %m%r; color label magenta;select 101:A and *.ca; label %m%r; color label magenta;select 102:A and *.ca; label %m%r; color label magenta;select 128:A and *.ca; label %m%r; color label magenta;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>

    <font face="Monospace"><td valign="middle" align="center">5</td><td valign="middle" align="center">0.04</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3cjq" target="_blank">3cjqD</a></font></td><td align="center">0.747</td><td valign="middle" align="center"> 2.65</td><td valign="middle" align="center">0.218</td><td valign="middle">0.902</td><td valign="middle" align="center">0.82</td><td>2MM</td><td><a href="model1_bs5.pdb"><font face="Arial">Download</font></a></td><td valign="middle">20,100,101,102,128</td></tr>

       </table>

       <br/>

       <table  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">

        <tr><td valign="top">&nbsp;</td><td align="justify"><b> Click on the radio buttons to visualize predicted binding site and residues.</b></td></tr>

        <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>LB</sup> is the confidence score of predicted binding site. Cscore<sup>LB</sup> values range in between [0-1]; where a higher score indicates a more reliable ligand-binding site prediction. </td></tr>

       <tr><td valign="top">(b)</td><td align="justify">BS-score is a measure of local similarity (sequence & structure) between template binding site and predicted binding site in the query structure. Based on large scale benchmarking analysis, we have observed that a BS-score >1 reflects a significant local match between the predicted and template binding site.

       <tr><td valign="top">(c)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>

       <tr><td valign="top">(d)</td><td align="justify">RMSD<sup>a</sup> the RMSD between residues that are structurally aligned by TM-align.</td></tr>

       <tr><td valign="top">(e)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>

       <tr><td valign="top">(f)</td><td align="justify">Cov. represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>

       </table></td></tr>

     </table>

   </td></tr></table>

  </div>

</td></tr>

<tr><td><br/><br/></td></tr>

<tr><td>

<h3 align="center">[Click on <a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S185852/S185852_results.tar.bz2">S185852_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page] </h3>

<hr><br/>

<div style="position:relative;left:20px;"><table style="font-family:Arial;font-size:14px;"><tr><td>&nbsp;</td><td>Please cite following articles when you use the I-TASSER server:</td><tr>

<tr><td valign="top">1.</td><td>Yang Zhang. I-TASSER server for protein 3D structure prediction.  BMC Bioinformatics, 9:40 (2008).</td></tr>

<tr><td valign="top">2.</td><td>Ambrish Roy, Alper Kucukural, Yang Zhang. I-TASSER: a unified platform for automated protein structure and function prediction. Nature Protocols, vol 5, 725-738 (2010).</td></tr>

<tr><td valign="top">3.</td><td>Ambrish Roy, Jianyi Yang, Yang Zhang. COFACTOR: an accurate comparative algorithm for structure-based protein function annotation. Nucleic Acids Research, vol 40, W471-W477 (2012).</td></tr>

</table></div></td></tr>

</table></body>

</html>

