<HTML>
<TITLE>LOOPP results</TITLE>
<BODY BGCOLOR="#FFFFFF" LINK="#0000FF" VLINK="#660099" ALINK="#660099"><PRE>
<b>LOOPP parallel driver v7.0 with LOOPP v3.20 Junior (compiled Nov 2010)</b>
     [NEWSABLE]     [HSHFL]     [ENE]     [NEWALICHOICE]

This is HTML output, the other option is <a href='aktm_2014_Jul_11.0001.out'><b>plain text</b></a> formats.

If an error occured during execution it is logged in plain text version only,
but no errors were ultimately fatal if this file ends with 'DONE' statement.

You can also view standard-error output for any warnings/messages
generated by externally used programs: <a href='loopp.err'><b>loopp.err</b></a>

<b>STARTING:</b>Fri Jul 11 10:51:57 2014 EST

<b>INPUT:</b>
   <i>sim_min</i> =  40.0% minimum sequence identity (%)
   <i>sim_max</i> =  80.0% maximum sequence identity (%)
   <i>len_dif</i> =  20.0% maximum length difference (% of query sequence length) 
                    between query and homolog
   <i>nmax</i>    =  1     maximum number of homologous sequences (including query sequence)
   <i>maxout</i>  =  5     maximum number of outputted matches
   <i>strict</i>  =  0     all alignments will be accepted from LOOPP results
                    for homologs; they will be translated to query sequence 
                    alignments using BLAST results
   <i>casp</i>    =  5     maximum number of outputted matches in CASP format
   <i>align</i>   =  'g'   only corresponding global alignments will be reported
   <i>maxdiff</i> =  50.0% only alignments differing no more than  50.0% from the query
                    sequence will be reported, or when reject=0, corresponding local
                    alignments will be reported
   <i>reject</i>  =  1     alignments shorter more than  50.0% from the query sequence
                    will be rejected
   <i>seccut</i>  =  0.00  structures with sec. str. score below seccut will be rejected
    <i>secdb</i>  = 1      type of secondary structure description: 1 - DSSB, 0 - phi/psi map
   <i>tasktb</i>  = 111111111111111111  type of initial calculations
   <i>pdbcut</i>  =  60.0% pdb sequence identity threshold (%)
  <i>pdbstop</i>  = 0      stop if PDB structure with similarity above looppOptions.pdbcut found
  <i>domains</i>  = 0      query sequence will be decomposed into putative domains if =1
   <i>maxall</i>  = 4000      maximum number of models evaluated in the second stage       
   <i>refstr</i>  = 0      if equal to 1 rmsd distances between all hits and a reference structure
                   will be reported
   <i>maxpdb</i>  = 5      number of top PDB hits added to the second stage scoring 
   <i>maxdyn</i>  = 5      number of dynamic domain entries from database added to the second stage scoring 
 <i>beautify</i>  = 0      use pdb beautifier 
<i>rmsdtables</i>  = 1     compute rmsd tables between hits 
<i>cacheblast</i>  = 0     use/save cached blast results 
<i>cachesable</i>  = 0     use/save cached sable results 
<i>noLocalDat</i>  = 0     all database accessed over network 
<i>optm_alignlen_cutoff</i>  = 0.7     fraction of querylen required for template alignments 
<i>p2_reduced_conf</i>  = 0     phase2 uses reduced confidence tree 

   <i>local</i> LOOPP/BLAST database will be used

   <i>NR    database</i> = /state/partition1/looppLocal//blastdb/20110524/nr
   <i>PDBAA database</i> = /state/partition1/looppLocal//blastdb/20110524/pdbaa
   input file <a href='aktm_2014_Jul_11.0001'><b><i>aktm_2014_Jul_11.0001</b></i></a>
    --> 163 residues, sequence name: <i>aktm_2014_Jul_11.0001</i>

5 dynamic database entries will be attempted:
  3CJT_A [success]
  1KXZ_A [success]
  3MGG_A [success]
  2BH2_A [success]
  1UWV_A [success]




 -> 66 <a href='aktm_2014_Jul_11.0001.out.blast'>homolog sequences</a> with sequence identity from  40.0% to  80.0% satisfying length constraints found
 -> 0 representatives will be used in search (nmax=1 including query sequence)
 Histogram of all homolog sequences found by BLAST (e-value cutoff 0.01, length constr.) by sequence identity:
<table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td>  00%-10%  </td><td>  10%-20%  </td><td>  20%-30%  </td><td>  30%-40%  </td><td>  40%-50%  </td><td>  50%-60%  </td><td>  60%-70%  </td><td>  70%-80%  </td><td>  80%-90%  </td><td>  90%-100% </td></tr><tr>
<td align='center'>       0   </td><td align='center'>       0   </td><td align='center'>      62   </td><td align='center'>      72   </td><td align='center'>      43   </td><td align='center'>      17   </td><td align='center'>       3   </td><td align='center'>       3   </td><td align='center'>       2   </td><td align='center'>       2   </td></tr></table>


<a href='protein1.2nd'>secondary structure prediction</a> shows 31.90% helical, 22.09% extended and 46.01% loops/other

<a href='aktm_2014_Jul_11.0001.homolog'>homologs/domains used:</a>
  0 query sequence <a href='protein1.seq'><i>aktm_2014_Jul_11.0001</i></a>
<a href='aktm_2014_Jul_11.0001.out.blast'>all homologs found</a>

total number of entries processed: 310

Amino acid profile calculated versus nr database is <a href='aktm_2014_Jul_11.0001.tblast.pos'>here</a>
  (Amino acid profile BLAST run output is <a href='aktm_2014_Jul_11.0001.tblast.out1'>here</a>)
Results of BLAST versus pdbaa database using the nr profile are <a href='aktm_2014_Jul_11.0001.tblast.out2'>here</a>
  (Results of BLAST versus pdbaa database with no profile are <a href='aktm_2014_Jul_11.0001.out.blastpdb'>here</a>)
Transitive BLAST output calculted with this profile is <a href='aktm_2014_Jul_11.0001.tblast.out2'>here</a>

<b>LOOPP uses several decision trees for identifying and scoring final models.  The results for
each tree are presented in order, from the most confident tree (LP9) to the least confident
tree (LP8_reduced_confidence).  The HITS under a given tree may be empty if that tree did
not identify any models within its confidence limits.</b>


<b>RESULTS for descision tree LP9</b>

Phase 1 scoring data: <a href='phase1list.in.1'>input</a> and <a href='phase1.out.1.LP9'>output</a>
Phase 2 scoring data: <a href='phase2.in.123456.LP9'>input</a> and <a href='phase2.out.123456.LP9'>output</a>


<a name='hits'><b>HITS:</b>

<a href="legend.htm" target=_blank><i>table legend</i></a>
Model  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                            (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='#hit1'>1  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BUS' target=_blank>3BUS_A</a> METHYLTRANSFERASE;                                      -17.220    0     0  87.12%  19.72%   OPTM     -438.777     -358.270        2.397        0.622        0.587        0.527        0.913     22.930     -49.8     1.420    46.000     120.5     4.690    52.500    -135.0     1.950    10.540     422.6     0.270     2.240     -50.7     1.090     7.390     429.1     2.230     5.980     -40.5     1.150     9.370     242.4    13.270    14.960     -66.4    -1.000    13.000     -60.0    -1.000    16.000    -118.0     3.600    90.500     162.5    15.720   101.000    -153.0     0.220    15.000      57.0     6.360    41.000    -114.0    19.220     -17.2     4.100      -2.1    19.220     -17.2    0.83  0.94  0.93
<a href='#hit2'>2  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UFK' target=_blank>1UFK_A</a> TT0836 PROTEIN;                                         -15.050    0     0  60.12%  13.51%   OPTM     -187.890     -182.070        1.261        0.436        0.281        0.292        0.560      3.490     -17.4    -0.440     0.000     180.0     8.360    83.500    -162.5    -1.500   -16.280     376.9    -1.750    -0.370     -42.0    -1.870   -13.230     394.4    -1.070     0.880     -32.2     2.420    -9.930     213.7     9.050     2.900     -54.0    -1.000    -4.000     -45.0    -1.000     8.000    -120.0    -0.860   -45.500     263.5    16.460   112.000    -161.0    -0.820     1.000     208.0    -0.010    -3.000     -22.0    17.050     -15.1     4.100      -2.1    17.000     -15.0   -0.13  0.70  0.76
<a href='#hit3'>3  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CJT' target=_blank>3CJT_A</a> RIBOSOMAL PROTEIN L11 METHYLTRANSFERASE;                -14.699    0     1  94.48%  22.93%   OPTM     -452.020     -306.830        2.147        0.604        0.593        0.589        0.948     17.040     -46.4    -0.800    -6.000     201.0     7.570    80.500    -152.0    -2.200    -9.290     451.4    -1.730    -2.260     -30.6    -1.880    -6.300     460.7     0.970     2.570     -24.7    -0.450   -17.630     258.5    15.590    19.600     -67.2   -30.000    -2.000     -46.0   -30.000    16.000    -124.0    -0.460   -29.500     272.5    15.660   110.000    -163.0    -1.530     3.000      88.0     5.590    31.000    -106.0     3.694     -14.7     4.581      -2.3     3.714     -14.7    0.57  0.75  0.76
<a href='#hit4'>4  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXZ' target=_blank>1KXZ_C</a> PRECORRIN-6Y METHYLTRANSFERASE/PUTATIVE                 -14.400    0     0 100.00%  19.63%   OPTM     -531.129     -303.340        3.568        0.584        0.536        0.822        0.935     27.580     -48.4     5.370   133.500    -119.5     9.060   105.000    -186.5    -1.100    -6.100      40.8     0.490    -0.470     -39.4     0.790     4.130      50.9     1.850     1.560     -32.8     6.090    38.330      -2.3    16.780    24.350     -68.6    -1.000    28.000    -109.0    -1.000    33.000    -127.0     8.940   172.500     -94.5    17.800   143.000    -183.0     4.950    48.000     -83.0     9.040    56.000    -133.0    16.400     -14.4     5.000      -3.0    16.520     -14.5    0.73  0.84  0.76
<a href='#hit5'>5  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXZ' target=_blank>1KXZ_A</a> PRECORRIN-6Y METHYLTRANSFERASE/PUTATIVE                 -14.398    0     1 100.00%  19.63%   OPTM     -462.269     -233.550        3.284        0.575        0.560        0.835        0.950     27.760     -48.5     5.440   135.000    -120.5     9.130   106.000    -187.0    -0.780    -2.180      42.6    -0.160    -1.970     -36.9     0.490     3.400      55.7     2.060     2.400     -33.0     5.640    34.570       0.9    17.210    23.930     -68.2   -30.000    28.000    -109.0   -30.000    33.000    -127.0     7.220   146.500     -62.5    15.770   113.000    -154.0     4.830    48.000     -82.0     9.120    56.000    -133.0     5.000     -14.4     4.284      -3.0     4.504     -14.5    0.73  0.84  0.76



<b>RMSD table between hits</b>
(RMSDs are shaded according to similarity, up to 10&#197 )
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#ffffff"><p align='right'>  14.18 </td><td bgcolor="#d3d3d3"><p align='right'>   5.65 </td><td bgcolor="#dbdbdb"><p align='right'>   6.46 </td><td bgcolor="#dadada"><p align='right'>   6.36 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#ffffff"><p align='right'>  14.18 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#ffffff"><p align='right'>  15.22 </td><td bgcolor="#ffffff"><p align='right'>  16.47 </td><td bgcolor="#ffffff"><p align='right'>  16.42 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#d3d3d3"><p align='right'>   5.65 </td><td bgcolor="#ffffff"><p align='right'>  15.22 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#cccccc"><p align='right'>   4.98 </td><td bgcolor="#c9c9c9"><p align='right'>   4.70 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#dbdbdb"><p align='right'>   6.46 </td><td bgcolor="#ffffff"><p align='right'>  16.47 </td><td bgcolor="#cccccc"><p align='right'>   4.98 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#bdbdbd"><p align='right'>   3.42 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#dadada"><p align='right'>   6.36 </td><td bgcolor="#ffffff"><p align='right'>  16.42 </td><td bgcolor="#c9c9c9"><p align='right'>   4.70 </td><td bgcolor="#bdbdbd"><p align='right'>   3.42 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td></tr>
</table><PRE>


<b>Similarity table for hits: percentage of chain fitting below 3&#197 RMSD</b>
(Cells are shaded according to similarity starting at 30%)
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  25.56 </td><td bgcolor="#cfcfcf"><p align='right'>  63.57 </td><td bgcolor="#b1b1b1"><p align='right'>  84.51 </td><td bgcolor="#b1b1b1"><p align='right'>  84.51 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#ffffff"><p align='right'>  25.56 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  18.69 </td><td bgcolor="#ffffff"><p align='right'>  18.92 </td><td bgcolor="#ffffff"><p align='right'>  18.92 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#cfcfcf"><p align='right'>  63.57 </td><td bgcolor="#ffffff"><p align='right'>  18.69 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#b2b2b2"><p align='right'>  83.44 </td><td bgcolor="#afafaf"><p align='right'>  85.35 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#b1b1b1"><p align='right'>  84.51 </td><td bgcolor="#ffffff"><p align='right'>  18.92 </td><td bgcolor="#b2b2b2"><p align='right'>  83.44 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'>  99.39 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#b1b1b1"><p align='right'>  84.51 </td><td bgcolor="#ffffff"><p align='right'>  18.92 </td><td bgcolor="#afafaf"><p align='right'>  85.35 </td><td bgcolor="#9b9b9b"><p align='right'>  99.39 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
</table><PRE>


<b>Similarity table for hits: percentage of chain fitting below 6&#197 RMSD</b>
(Cells are shaded according to similarity starting at 50%)
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  35.56 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9d9d9d"><p align='right'>  98.59 </td><td bgcolor="#9d9d9d"><p align='right'>  98.59 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#ffffff"><p align='right'>  35.56 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  29.91 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  29.91 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#9d9d9d"><p align='right'>  98.59 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#9d9d9d"><p align='right'>  98.59 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
</table><PRE>




<b>ALIGNMENTS</b>

<a name='hit1'><b>--> hit 1</b> <a href='models_LP9/model_all001.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 34.16% helical, 22.98% extended, 42.86% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BUS' target=_blank>3BUS_A</a> METHYLTRANSFERASE;                                      -17.220    0     0  87.12%  19.72%   OPTM     -438.777     -358.270        2.397        0.622        0.587        0.527        0.913     22.930     -49.8     1.420    46.000     120.5     4.690    52.500    -135.0     1.950    10.540     422.6     0.270     2.240     -50.7     1.090     7.390     429.1     2.230     5.980     -40.5     1.150     9.370     242.4    13.270    14.960     -66.4    -1.000    13.000     -60.0    -1.000    16.000    -118.0     3.600    90.500     162.5    15.720   101.000    -153.0     0.220    15.000      57.0     6.360    41.000    -114.0    19.220     -17.2     4.100      -2.1    19.220     -17.2    0.83  0.94  0.93

alignment source: OPTM
            -------------          HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHH
Query:    1 MSYVPHVPYVPTP----------EKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALA   70
Sbjct:    2 WGENLHFGYWEDAGADVSVDDATDRLTDEMIALLDVRSGDRVLDVGCGIGKPAVRLATARDV-RVTGISISRPQVNQANA   80
            -----------------HHHHHHHHHHHHHHH-------EEEEE-----HHHHHHHHH---E EEEEE--HHHHHHHHHH

            HHHH------EEEEE-----------EEEEEEE--     HHHHHHHHHHHHH-----EEEEE---- ----  HHHHHH
Query:   71 NIEKNGVTGRASIVKGNFFEVDISEATVVTMFLLT-----NVNEMLKPKLEKELKPGTRVVSHEFEI-RGWN--PKEVIK  142
Sbjct:   81 RATAAGLANRVTFSYADAMDLPFEDASFDAVWALESLHHMPDRGRALREMARVLRPGGTVAIADFVLLAPVEGAKKEAVD  160
            HHH------EEEEE------------EEEEEEE--------HHHHHHHHH--EEEEEEEEEEEEEE-----HHHHHHHHH

142 residues (87.12%) of query sequence aligned

<HR>

<a name='hit2'><b>--> hit 2</b> <a href='models_LP9/model_all002.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 22.69% helical, 16.20% extended, 61.11% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UFK' target=_blank>1UFK_A</a> TT0836 PROTEIN;                                         -15.050    0     0  60.12%  13.51%   OPTM     -187.890     -182.070        1.261        0.436        0.281        0.292        0.560      3.490     -17.4    -0.440     0.000     180.0     8.360    83.500    -162.5    -1.500   -16.280     376.9    -1.750    -0.370     -42.0    -1.870   -13.230     394.4    -1.070     0.880     -32.2     2.420    -9.930     213.7     9.050     2.900     -54.0    -1.000    -4.000     -45.0    -1.000     8.000    -120.0    -0.860   -45.500     263.5    16.460   112.000    -161.0    -0.820     1.000     208.0    -0.010    -3.000     -22.0    17.050     -15.1     4.100      -2.1    17.000     -15.0   -0.13  0.70  0.76

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP9/model_all002.pdb.log'>here for details</a>
            -EEEEEE--HHHHHHHHHHHHH---      ---EEEEE-----------EEEEEEE--HHHHHHHHHHHHH-----E  
Query:   53 KKAVGVEINDERIREALANIEKNGV------TGRASIVKGNFFEVDISEATVVTMFLLTNVNEMLKPKLEKELKPGTR--  124
Sbjct:    1 MWVYRLKGTLEALDPILPGLFDGGARGLWEREGEVWAFFPA--PVDLPYEGVWEEVG----DEDWLEAWRRDLKPALAPP   74
            -EEEEEE--HHHH---HHHHHH----EEEEE--EEEEEE--  --------EEEE--    HHHHHHHHHHH---EEE--

            EEEE--------HHHHHHE-------EEEEEEE------
Query:  125 VVSHEFEIRGWNPKEVIKVEDGNMNHTVYLYVIGEHKLE  163
Sbjct:   75 FVVLAP-WHTW---------EGAE-----IPLVIEPGMA   98
            EEEE-- ----         ----     EEEE------

98 residues (60.12%) of query sequence aligned

<HR>

<a name='hit3'><b>--> hit 3</b> <a href='models_LP9/model_all003.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 23.48% helical, 17.81% extended, 58.70% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CJT' target=_blank>3CJT_A</a> RIBOSOMAL PROTEIN L11 METHYLTRANSFERASE;                -14.699    0     1  94.48%  22.93%   OPTM     -452.020     -306.830        2.147        0.604        0.593        0.589        0.948     17.040     -46.4    -0.800    -6.000     201.0     7.570    80.500    -152.0    -2.200    -9.290     451.4    -1.730    -2.260     -30.6    -1.880    -6.300     460.7     0.970     2.570     -24.7    -0.450   -17.630     258.5    15.590    19.600     -67.2   -30.000    -2.000     -46.0   -30.000    16.000    -124.0    -0.460   -29.500     272.5    15.660   110.000    -163.0    -1.530     3.000      88.0     5.590    31.000    -106.0     3.694     -14.7     4.581      -2.3     3.714     -14.7    0.57  0.75  0.76

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP9/model_all003.pdb.log'>here for details</a>
            -----------HHHHHHHHHH--  -----EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH------
Query:    3 YVPHVPYVPTPEKVVRRMLEIAK--VSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGR   80
Sbjct:   93 IEPGMAFGTGHAETTRLALKALARHLRPGDKVLDLGTGSG-VLAIAAEKLGG-KALGVDIDPMVLPQAEANAKRNGV--R  168
            -----------HHHHHHHHHHHHH-----EEEEE-----H HHHHHHH---E EEEEE-----HHHHHHHHH-----  -

            EEEEE------ -----EEEEEEE--HHHHHHHHHHHHH-----EEEEE--------HHHHHHE-------EEEEEEE--
Query:   81 ASIVKGNFFEV-DISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEIRGWNPKEVIKVEDGNMNHTVYLYVIGE  159
Sbjct:  169 PRFLEGSLEAALPFGPFDLLVANLYAELHAALAPRYREALVPGGRALLTGILKD--RAPLVREAMAGAGFRPLEEAAEGE  246
            EEEE--HHHH-----EEEEEEE--HHHHHHHHHHHHHHEEEEEEEEEEEEE---  HHHHHHHHHH---EEEEEEEE--E

154 residues (94.48%) of query sequence aligned

<HR>

<a name='hit4'><b>--> hit 4</b> <a href='models_LP9/model_all004.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 35.88% helical, 25.88% extended, 38.24% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXZ' target=_blank>1KXZ_C</a> PRECORRIN-6Y METHYLTRANSFERASE/PUTATIVE                 -14.400    0     0 100.00%  19.63%   OPTM     -531.129     -303.340        3.568        0.584        0.536        0.822        0.935     27.580     -48.4     5.370   133.500    -119.5     9.060   105.000    -186.5    -1.100    -6.100      40.8     0.490    -0.470     -39.4     0.790     4.130      50.9     1.850     1.560     -32.8     6.090    38.330      -2.3    16.780    24.350     -68.6    -1.000    28.000    -109.0    -1.000    33.000    -127.0     8.940   172.500     -94.5    17.800   143.000    -183.0     4.950    48.000     -83.0     9.040    56.000    -133.0    16.400     -14.4     5.000      -3.0    16.520     -14.5    0.73  0.84  0.76

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP9/model_all004.pdb.log'>here for details</a>
            ----  ---------HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH----
Query:    1 MSYV--PHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVT   78
Sbjct:    5 DEFIKNPSVPG-PTAMEVRCLIMCLAEPGKNDVAVDVGCGTGGVTLELAG--RVRRVYAIDRNPEAISTTEMNLQRHGLG   81
            ----------- --HHHHHHHHHHH------EEEEE-----HHHHHHH--  --EEEEEE--HHHHHHHHHHHHH-----

            --EEEEE-----  ------EEEEEEE--HHHHHHHHHHHHH-----EEEEE---- ----HHHHHHE-------EEEEE
Query:   79 GRASIVKGNFFE--VDISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEI-RGWNPKEVIKVEDGNMNHTVYLY  155
Sbjct:   82 DNVTLMEGDAPEALCKIPDIDIAVVGGSGGELQEILRIIKDKLKPGGRIIVTAILLETKFEAMECLRDLGFDVNITELNI  161
            -EEEEE--HHHHHH----EEEEEE------HHHHHHHHHHHEEEEEEEEEEE--HHHHHHHHHHHHH----EEEEEE--E

            EE------
Query:  156 VIGEHKLE  163
Sbjct:  162 ARGRALDR  169
            EEEE----

163 residues (100.00%) of query sequence aligned

<HR>

<a name='hit5'><b>--> hit 5</b> <a href='models_LP9/model_all005.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 35.88% helical, 25.88% extended, 38.24% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXZ' target=_blank>1KXZ_A</a> PRECORRIN-6Y METHYLTRANSFERASE/PUTATIVE                 -14.398    0     1 100.00%  19.63%   OPTM     -462.269     -233.550        3.284        0.575        0.560        0.835        0.950     27.760     -48.5     5.440   135.000    -120.5     9.130   106.000    -187.0    -0.780    -2.180      42.6    -0.160    -1.970     -36.9     0.490     3.400      55.7     2.060     2.400     -33.0     5.640    34.570       0.9    17.210    23.930     -68.2   -30.000    28.000    -109.0   -30.000    33.000    -127.0     7.220   146.500     -62.5    15.770   113.000    -154.0     4.830    48.000     -82.0     9.120    56.000    -133.0     5.000     -14.4     4.284      -3.0     4.504     -14.5    0.73  0.84  0.76

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP9/model_all005.pdb.log'>here for details</a>
            ----  ---------HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH----
Query:    1 MSYV--PHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVT   78
Sbjct:    5 DEFIKNPSVPG-PTAMEVRCLIMCLAEPGKNDVAVDVGCGTGGVTLELAG--RVRRVYAIDRNPEAISTTEMNLQRHGLG   81
            ----------- --HHHHHHHHHHH------EEEEE-----HHHHHHH--  --EEEEEE--HHHHHHHHHHHHH-----

            --EEEEE-----  ------EEEEEEE--HHHHHHHHHHHHH-----EEEEE---- ----HHHHHHE-------EEEEE
Query:   79 GRASIVKGNFFE--VDISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEI-RGWNPKEVIKVEDGNMNHTVYLY  155
Sbjct:   82 DNVTLMEGDAPEALCKIPDIDIAVVGGSGGELQEILRIIKDKLKPGGRIIVTAILLETKFEAMECLRDLGFDVNITELNI  161
            -EEEEE--HHHHHH----EEEEEE------HHHHHHHHHHHEEEEEEEEEEE--HHHHHHHHHHHHH----EEEEEE--E

            EE------
Query:  156 VIGEHKLE  163
Sbjct:  162 ARGRALDR  169
            EEEE----

163 residues (100.00%) of query sequence aligned

<HR>

<b>RESULTS for descision tree LP8</b>

Phase 1 scoring data: <a href='phase1list.in.1'>input</a> and <a href='phase1.out.1.LP8'>output</a>
Phase 2 scoring data: <a href='phase2.in.123456.LP8'>input</a> and <a href='phase2.out.123456.LP8'>output</a>


<a name='hits'><b>HITS:</b>

<a href="legend.htm" target=_blank><i>table legend</i></a>
Model  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                            (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='#hit1'>1  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IM8' target=_blank>1IM8_B</a> YECO;                                                   -41.752    0     0  81.60%  22.56%   OPTM     -492.361     -388.850        2.625        0.673        0.655        0.573        0.946     22.330     -47.4     5.270   119.500     -25.5     6.500    84.000    -155.0     1.370     2.080     251.4    -0.570     0.570     -33.2     3.630    20.470     248.1     2.040    10.080     -27.3     8.100    28.100     115.2    12.530     6.080     -56.7    -1.000     8.000     -77.0    -1.000    -4.000    -109.0     4.930    76.500      66.0    11.300    78.000    -125.0     3.200    20.000     -18.0     8.510    39.000    -121.0    10.100      -8.1     2.390      -0.4    10.100      -8.1   -0.10  0.90  0.75
<a href='#hit2'>2  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2V74' target=_blank>2V74_F</a> HISTONE-ARGININE METHYLTRANSFERASE CARM1;               -41.887    0     0  69.33%  25.66%   OPTM     -393.352     -218.580        1.323        0.651        0.378        0.322        0.930     19.480     -38.5     1.660    60.000     501.0     6.530    86.500    -152.5    -0.910     1.440    1169.9    -1.160    -4.310     -40.8    -1.070    -5.320    1188.5    -0.870    -8.010     -27.4     0.730     5.310     660.7     9.280    10.010     -52.5    -1.000    15.000      27.0    -1.000    -4.000    -110.0     0.180    25.000     598.0    11.080    60.000    -106.0     0.730    -5.000     252.0     5.110    27.000    -117.0     8.520      -6.5     3.250      -1.2     8.400      -6.4   -0.10  0.82  0.68
<a href='#hit3'>3  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3JWH' target=_blank>3JWH_A</a> HEN1;                                                   -41.900    0     0  70.55%  18.26%   OPTM     -491.079     -284.910        1.671        0.648        0.521        0.593        0.977     17.570     -38.1     1.790    47.000     -17.0     3.440    50.500    -125.0     0.740    -0.200      30.1    -1.640    -0.660     -47.1     1.540     1.790      52.3    -0.850    -2.130     -34.7    -2.170   -26.900      40.8    11.020    14.080     -61.0    -1.000     7.000     -85.0    -1.000    10.000    -112.0     1.950    62.500      44.0     5.130    41.000     -86.0    -0.460    -5.000      10.0     2.590    13.000     -78.0     5.400      -3.4     1.540       0.5     5.300      -3.3   -0.10  0.91  0.44
<a href='#hit4'>4  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DLC' target=_blank>3DLC_A</a> PUTATIVE S-ADENOSYL-L-METHIONINE-DEPENDENT              -41.937    0     0  82.21%  17.04%   OPTM     -527.006     -370.390        2.829        0.662        0.580        0.577        0.928     18.040     -42.7     1.550    38.500      26.5     2.880     9.500    -113.5     1.820    12.490     224.1     0.430     4.340     -33.0     2.040    19.810     229.5     0.400     3.210     -23.1     6.500    24.510     109.2    14.490    11.310     -56.6    -1.000    19.000     -87.0    -1.000     8.000    -111.0     5.810   125.500      19.0    11.870    84.000    -137.0     4.400    31.000     -38.0    10.040    48.000    -138.0     8.050      -6.0     0.000       2.0     8.000      -6.0    0.00  0.94  0.75
<a href='#hit5'>5  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ESR' target=_blank>2ESR_A</a> METHYLTRANSFERASE;                                      -41.959    0     0  89.57%  20.55%   OPTM     -462.696     -459.320        1.767        0.645        0.517        0.891        0.975     14.950     -35.2     1.300    17.000     -34.5     1.220    16.000     -83.0    -0.310     3.630      -9.3    -0.960     2.830     -30.7    -0.160     6.310       9.4    -1.540     0.870     -22.9    -1.250    -7.760      -1.5    12.200    16.180     -53.6    -1.000    19.000     -93.0    -1.000    25.000    -108.0     3.480    80.000     -10.0     4.960    27.000     -72.0     2.360    28.000     -44.0     4.680    30.000     -80.0     3.300      -1.3     0.000       2.0     5.000      -3.0    0.14  0.75  0.35



<b>RMSD table between hits</b>
(RMSDs are shaded according to similarity, up to 10&#197 )
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#bfbfbf"><p align='right'>   3.68 </td><td bgcolor="#b6b6b6"><p align='right'>   2.76 </td><td bgcolor="#d5d5d5"><p align='right'>   5.88 </td><td bgcolor="#dddddd"><p align='right'>   6.61 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#bfbfbf"><p align='right'>   3.68 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#bbbbbb"><p align='right'>   3.30 </td><td bgcolor="#b5b5b5"><p align='right'>   2.68 </td><td bgcolor="#b4b4b4"><p align='right'>   2.54 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#b6b6b6"><p align='right'>   2.76 </td><td bgcolor="#bbbbbb"><p align='right'>   3.30 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#b8b8b8"><p align='right'>   2.97 </td><td bgcolor="#bababa"><p align='right'>   3.19 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#d5d5d5"><p align='right'>   5.88 </td><td bgcolor="#b5b5b5"><p align='right'>   2.68 </td><td bgcolor="#b8b8b8"><p align='right'>   2.97 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#dcdcdc"><p align='right'>   6.59 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#dddddd"><p align='right'>   6.61 </td><td bgcolor="#b4b4b4"><p align='right'>   2.54 </td><td bgcolor="#bababa"><p align='right'>   3.19 </td><td bgcolor="#dcdcdc"><p align='right'>   6.59 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td></tr>
</table><PRE>


<b>Similarity table for hits: percentage of chain fitting below 3&#197 RMSD</b>
(Cells are shaded according to similarity starting at 30%)
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#bebebe"><p align='right'>  75.22 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#d1d1d1"><p align='right'>  61.65 </td><td bgcolor="#c3c3c3"><p align='right'>  72.00 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#bebebe"><p align='right'>  75.22 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#b1b1b1"><p align='right'>  84.07 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#b1b1b1"><p align='right'>  84.07 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#a5a5a5"><p align='right'>  92.79 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#d1d1d1"><p align='right'>  61.65 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#a7a7a7"><p align='right'>  91.20 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#c3c3c3"><p align='right'>  72.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#a5a5a5"><p align='right'>  92.79 </td><td bgcolor="#a7a7a7"><p align='right'>  91.20 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
</table><PRE>


<b>Similarity table for hits: percentage of chain fitting below 6&#197 RMSD</b>
(Cells are shaded according to similarity starting at 50%)
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9e9e9e"><p align='right'>  98.40 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9e9e9e"><p align='right'>  98.40 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#9e9e9e"><p align='right'>  98.40 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9e9e9e"><p align='right'>  98.40 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
</table><PRE>




<b>ALIGNMENTS</b>

<a name='hit1'><b>--> hit 1</b> <a href='models_LP8/model_all001.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 35.62% helical, 23.12% extended, 41.25% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IM8' target=_blank>1IM8_B</a> YECO;                                                   -41.752    0     0  81.60%  22.56%   OPTM     -492.361     -388.850        2.625        0.673        0.655        0.573        0.946     22.330     -47.4     5.270   119.500     -25.5     6.500    84.000    -155.0     1.370     2.080     251.4    -0.570     0.570     -33.2     3.630    20.470     248.1     2.040    10.080     -27.3     8.100    28.100     115.2    12.530     6.080     -56.7    -1.000     8.000     -77.0    -1.000    -4.000    -109.0     4.930    76.500      66.0    11.300    78.000    -125.0     3.200    20.000     -18.0     8.510    39.000    -121.0    10.100      -8.1     2.390      -0.4    10.100      -8.1   -0.10  0.90  0.75

alignment source: OPTM
            ----HHHHHHHHHH--  -----EEEEE----HHHHHHHHHH----  EEEEEE--HHHHHHHHHHHHH------EEEEE
Query:   10 VPTPEKVVRRMLEIAK--VSQDDIVYDLGCGDGRIIITAAKDFNVK--KAVGVEINDERIREALANIEKNGVTGRASIVK   85
Sbjct:   17 VPGYSNIITAIGMLAERFVTADSNVYDLGCSRGAATLSARRNINQPNVKIIGIDNSQPMVERCRQHIAAYHSEIPVEILC   96
            --HHHHHHHHHHHHHHH----EEEEEE-----HHHHHHH-------EEEEEEE--HHHHHHHHHHHH-------EEEEE-

            -----------EEEEEEE--     HHHHHHHHHHHHH--- --EEEEE--------HHHHHH
Query:   86 GNFFEVDISEATVVTMFLLT-----NVNEMLKPKLEKELKP-GTRVVSHEFEIRGWNPKEVIK  142
Sbjct:   97 NDIRHVEIKNASMVILNFTLQFLPPEDRIALLTKIYEGLNPNGVLVLSEKFRFEDTKINHLLI  159
            ---------EEEEEEE-------HHHHHHHHHHHHHHEEEEEEEEEEEE-----HHHHHHHHH

133 residues (81.60%) of query sequence aligned

<HR>

<a name='hit2'><b>--> hit 2</b> <a href='models_LP8/model_all002.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 23.94% helical, 21.13% extended, 54.93% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2V74' target=_blank>2V74_F</a> HISTONE-ARGININE METHYLTRANSFERASE CARM1;               -41.887    0     0  69.33%  25.66%   OPTM     -393.352     -218.580        1.323        0.651        0.378        0.322        0.930     19.480     -38.5     1.660    60.000     501.0     6.530    86.500    -152.5    -0.910     1.440    1169.9    -1.160    -4.310     -40.8    -1.070    -5.320    1188.5    -0.870    -8.010     -27.4     0.730     5.310     660.7     9.280    10.010     -52.5    -1.000    15.000      27.0    -1.000    -4.000    -110.0     0.180    25.000     598.0    11.080    60.000    -106.0     0.730    -5.000     252.0     5.110    27.000    -117.0     8.520      -6.5     3.250      -1.2     8.400      -6.4   -0.10  0.82  0.68

alignment source: OPTM
            HHHHHHHHHH---- ---EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH------EEEEE-------
Query:   14 EKVVRRMLEIAKVS-QDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGRASIVKGNFFEVD   92
Sbjct:   24 TGTYQRAILQNHTDFKDKIVLDVGCGSGILSFFAAQAG-ARKIYAVEAS-TMAQHAEVLVKSNNLTDRIVVIPGKVEEVS  101
            HHHHHHHHH--------EEEEE-----HHHHHHHH--- -EEEEEE--- -HHHHHHHHHH------EEEEE--------

            ----EEEEEEE--         HHHHHHHHHHHHH-----EEE
Query:   93 ISEATVVTMFLLT---------NVNEMLKPKLEKELKPGTRVV  126
Sbjct:  102 LPE-QVDIIISEPMGYMLFNERMLESYLHAK--KYLKPSGNMF  141
            --- EEEEEE-----------HHHHHHH---  -EEEEEEEE-

113 residues (69.33%) of query sequence aligned

<HR>

<a name='hit3'><b>--> hit 3</b> <a href='models_LP8/model_all003.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 35.88% helical, 21.37% extended, 42.75% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3JWH' target=_blank>3JWH_A</a> HEN1;                                                   -41.900    0     0  70.55%  18.26%   OPTM     -491.079     -284.910        1.671        0.648        0.521        0.593        0.977     17.570     -38.1     1.790    47.000     -17.0     3.440    50.500    -125.0     0.740    -0.200      30.1    -1.640    -0.660     -47.1     1.540     1.790      52.3    -0.850    -2.130     -34.7    -2.170   -26.900      40.8    11.020    14.080     -61.0    -1.000     7.000     -85.0    -1.000    10.000    -112.0     1.950    62.500      44.0     5.130    41.000     -86.0    -0.460    -5.000      10.0     2.590    13.000     -78.0     5.400      -3.4     1.540       0.5     5.300      -3.3   -0.10  0.91  0.44

alignment source: OPTM
            HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH----    --EEEEE----
Query:   14 EKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVT----GRASIVKGNFF   89
Sbjct:    5 QQRMNGVVAALKQSNARRVIDLGCGQGNLLKILLKDSFFEQITGVDVSYRSLEIAQERLDRLRLPRNQWERLQLIQGALT   84
            HHHHHHHHHHHH----EEEEE-----HHHHHHHH-----EEEEEE--HHHHHHHHHHH------HHHH--EEEEE-----

            ---  ----EEEEEEE--     HHHHHHHHHHHHH-----EEEEE
Query:   90 EVD--ISEATVVTMFLLT-----NVNEMLKPKLEKELKPGTRVVSH  128
Sbjct:   85 YQDKRFHGYDAATVIEVIEHLDLSRLGAFERVLFEFAQPKIVIVTT  130
            ---------EEEEE-------HHHHHHHHHHHH------EEEEEEE

115 residues (70.55%) of query sequence aligned

<HR>

<a name='hit4'><b>--> hit 4</b> <a href='models_LP8/model_all004.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 32.91% helical, 24.05% extended, 43.04% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DLC' target=_blank>3DLC_A</a> PUTATIVE S-ADENOSYL-L-METHIONINE-DEPENDENT              -41.937    0     0  82.21%  17.04%   OPTM     -527.006     -370.390        2.829        0.662        0.580        0.577        0.928     18.040     -42.7     1.550    38.500      26.5     2.880     9.500    -113.5     1.820    12.490     224.1     0.430     4.340     -33.0     2.040    19.810     229.5     0.400     3.210     -23.1     6.500    24.510     109.2    14.490    11.310     -56.6    -1.000    19.000     -87.0    -1.000     8.000    -111.0     5.810   125.500      19.0    11.870    84.000    -137.0     4.400    31.000     -38.0    10.040    48.000    -138.0     8.050      -6.0     0.000       2.0     8.000      -6.0    0.00  0.94  0.75

alignment source: OPTM
            ------HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH------EEEEE--
Query:    8 PYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGRASIVKGN   87
Sbjct:   24 LFAPIYPIIAENIINRFGIT-AGTCIDIGSGPGALSIALAKQSDF-SIRALDFSKHMNEIALKNIADANLNDRIQIVQGD  101
            ---HHHHHHHHHHHHH---E EEEEEE-----HHHHHHHHH-EEE EEEEE--HHHHHHHHHHHHH------EEEEE---

            ---------EEEEEEE--HH     HHHHHHHHHHH-----EEEEE--------HHHHHH
Query:   88 FFEVDISEATVVTMFLLTNV-----NEMLKPKLEKELKPGTRVVSHEFEIRGWNPKEVIK  142
Sbjct:  102 VHNIPIEDNYADLIVSRGSVFFWEDVATAFREIYRILKSGGKTYIGGGFGN----KELRD  157
            ---------EEEEEEE--------HHHHHHHHHHHEEEEEEEEEEE----H    HHHHH

134 residues (82.21%) of query sequence aligned

<HR>

<a name='hit5'><b>--> hit 5</b> <a href='models_LP8/model_all005.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 29.56% helical, 32.08% extended, 38.36% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ESR' target=_blank>2ESR_A</a> METHYLTRANSFERASE;                                      -41.959    0     0  89.57%  20.55%   OPTM     -462.696     -459.320        1.767        0.645        0.517        0.891        0.975     14.950     -35.2     1.300    17.000     -34.5     1.220    16.000     -83.0    -0.310     3.630      -9.3    -0.960     2.830     -30.7    -0.160     6.310       9.4    -1.540     0.870     -22.9    -1.250    -7.760      -1.5    12.200    16.180     -53.6    -1.000    19.000     -93.0    -1.000    25.000    -108.0     3.480    80.000     -10.0     4.960    27.000     -72.0     2.360    28.000     -44.0     4.680    30.000     -80.0     3.300      -1.3     0.000       2.0     5.000      -3.0    0.14  0.75  0.35

alignment source: OPTM
            HHHHHH- ------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH------EEEEE--- -------
Query:   18 RRMLEIA-KVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGRASIVKGNF-FEVDISE   95
Sbjct:    3 GAIFNMIGPYFNGGRVLDLFAGSGGLAIEAVSRG-MSAAVLVEKNRKAQAIIQDNIIMTKAENRFTLLKMEAERAIDCLT   81
            HHHHHH-------EEEEE-----HHHHHHHH--- -EEEEE---HHHHHHHHHHHH-------EEEE---HHHHHHH---

            -EEEEEEE--     HHHHHHHHHHHHH-----EEEEE--------HHHHHHE-------E    EEEEEE------
Query:   96 ATVVTMFLLT-----NVNEMLKPKLEKELKPGTRVVSHEFEIRGWNPKEVIKVEDGNMNHT----VYLYVIGEHKLE  163
Sbjct:   82 GRFDLVFLDPPYAKETIVATIEALAAKNLLSEQVMVVCETDKTVLLPKEIATLGIWKEKIYGISKVTVYVNEGHHHH  158
            -EEEEEE----HHHHHHHHHHHHHH---EEEEEEEEEEEE-------EE--EEEEEEEEE--EEEEEEEE-------

146 residues (89.57%) of query sequence aligned

<HR>

<b>RESULTS for descision tree LP9_redconf</b>

Phase 1 scoring data: <a href='phase1list.in.1'>input</a> and <a href='phase1.out.1.LP9'>output</a>
Phase 2 scoring data: <a href='phase2.in.123456.LP9_redconf'>input</a> and <a href='phase2.out.123456.LP9_redconf'>output</a>


<a name='hits'><b>HITS:</b>

<a href="legend.htm" target=_blank><i>table legend</i></a>
Model  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                            (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='#hit1'>1  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BUS' target=_blank>3BUS_A</a> METHYLTRANSFERASE;                                      -17.220    0     0  87.12%  19.72%   OPTM     -438.777     -358.270        2.397        0.622        0.587        0.527        0.913     22.930     -49.8     1.420    46.000     120.5     4.690    52.500    -135.0     1.950    10.540     422.6     0.270     2.240     -50.7     1.090     7.390     429.1     2.230     5.980     -40.5     1.150     9.370     242.4    13.270    14.960     -66.4    -1.000    13.000     -60.0    -1.000    16.000    -118.0     3.600    90.500     162.5    15.720   101.000    -153.0     0.220    15.000      57.0     6.360    41.000    -114.0    19.220     -17.2     4.100      -2.1    19.220     -17.2    0.83  0.94  0.93
<a href='#hit2'>2  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UFK' target=_blank>1UFK_A</a> TT0836 PROTEIN;                                         -15.050    0     0  60.12%  13.51%   OPTM     -187.890     -182.070        1.261        0.436        0.281        0.292        0.560      3.490     -17.4    -0.440     0.000     180.0     8.360    83.500    -162.5    -1.500   -16.280     376.9    -1.750    -0.370     -42.0    -1.870   -13.230     394.4    -1.070     0.880     -32.2     2.420    -9.930     213.7     9.050     2.900     -54.0    -1.000    -4.000     -45.0    -1.000     8.000    -120.0    -0.860   -45.500     263.5    16.460   112.000    -161.0    -0.820     1.000     208.0    -0.010    -3.000     -22.0    17.050     -15.1     4.100      -2.1    17.000     -15.0   -0.13  0.70  0.76
<a href='#hit3'>3  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CJT' target=_blank>3CJT_A</a> RIBOSOMAL PROTEIN L11 METHYLTRANSFERASE;                -14.699    0     1  94.48%  22.93%   OPTM     -452.020     -306.830        2.147        0.604        0.593        0.589        0.948     17.040     -46.4    -0.800    -6.000     201.0     7.570    80.500    -152.0    -2.200    -9.290     451.4    -1.730    -2.260     -30.6    -1.880    -6.300     460.7     0.970     2.570     -24.7    -0.450   -17.630     258.5    15.590    19.600     -67.2   -30.000    -2.000     -46.0   -30.000    16.000    -124.0    -0.460   -29.500     272.5    15.660   110.000    -163.0    -1.530     3.000      88.0     5.590    31.000    -106.0     3.694     -14.7     4.581      -2.3     3.714     -14.7    0.57  0.75  0.76
<a href='#hit4'>4  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXZ' target=_blank>1KXZ_C</a> PRECORRIN-6Y METHYLTRANSFERASE/PUTATIVE                 -14.400    0     0 100.00%  19.63%   OPTM     -531.129     -303.340        3.568        0.584        0.536        0.822        0.935     27.580     -48.4     5.370   133.500    -119.5     9.060   105.000    -186.5    -1.100    -6.100      40.8     0.490    -0.470     -39.4     0.790     4.130      50.9     1.850     1.560     -32.8     6.090    38.330      -2.3    16.780    24.350     -68.6    -1.000    28.000    -109.0    -1.000    33.000    -127.0     8.940   172.500     -94.5    17.800   143.000    -183.0     4.950    48.000     -83.0     9.040    56.000    -133.0    16.400     -14.4     5.000      -3.0    16.520     -14.5    0.73  0.84  0.76
<a href='#hit5'>5  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXZ' target=_blank>1KXZ_A</a> PRECORRIN-6Y METHYLTRANSFERASE/PUTATIVE                 -14.398    0     1 100.00%  19.63%   OPTM     -462.269     -233.550        3.284        0.575        0.560        0.835        0.950     27.760     -48.5     5.440   135.000    -120.5     9.130   106.000    -187.0    -0.780    -2.180      42.6    -0.160    -1.970     -36.9     0.490     3.400      55.7     2.060     2.400     -33.0     5.640    34.570       0.9    17.210    23.930     -68.2   -30.000    28.000    -109.0   -30.000    33.000    -127.0     7.220   146.500     -62.5    15.770   113.000    -154.0     4.830    48.000     -82.0     9.120    56.000    -133.0     5.000     -14.4     4.284      -3.0     4.504     -14.5    0.73  0.84  0.76



<b>RMSD table between hits</b>
(RMSDs are shaded according to similarity, up to 10&#197 )
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#ffffff"><p align='right'>  14.18 </td><td bgcolor="#d3d3d3"><p align='right'>   5.65 </td><td bgcolor="#dbdbdb"><p align='right'>   6.46 </td><td bgcolor="#dadada"><p align='right'>   6.36 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#ffffff"><p align='right'>  14.18 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#ffffff"><p align='right'>  15.22 </td><td bgcolor="#ffffff"><p align='right'>  16.47 </td><td bgcolor="#ffffff"><p align='right'>  16.42 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#d3d3d3"><p align='right'>   5.65 </td><td bgcolor="#ffffff"><p align='right'>  15.22 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#cccccc"><p align='right'>   4.98 </td><td bgcolor="#c9c9c9"><p align='right'>   4.70 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#dbdbdb"><p align='right'>   6.46 </td><td bgcolor="#ffffff"><p align='right'>  16.47 </td><td bgcolor="#cccccc"><p align='right'>   4.98 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#bdbdbd"><p align='right'>   3.42 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#dadada"><p align='right'>   6.36 </td><td bgcolor="#ffffff"><p align='right'>  16.42 </td><td bgcolor="#c9c9c9"><p align='right'>   4.70 </td><td bgcolor="#bdbdbd"><p align='right'>   3.42 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td></tr>
</table><PRE>


<b>Similarity table for hits: percentage of chain fitting below 3&#197 RMSD</b>
(Cells are shaded according to similarity starting at 30%)
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  25.56 </td><td bgcolor="#cfcfcf"><p align='right'>  63.57 </td><td bgcolor="#b1b1b1"><p align='right'>  84.51 </td><td bgcolor="#b1b1b1"><p align='right'>  84.51 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#ffffff"><p align='right'>  25.56 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  18.69 </td><td bgcolor="#ffffff"><p align='right'>  18.92 </td><td bgcolor="#ffffff"><p align='right'>  18.92 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#cfcfcf"><p align='right'>  63.57 </td><td bgcolor="#ffffff"><p align='right'>  18.69 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#b2b2b2"><p align='right'>  83.44 </td><td bgcolor="#afafaf"><p align='right'>  85.35 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#b1b1b1"><p align='right'>  84.51 </td><td bgcolor="#ffffff"><p align='right'>  18.92 </td><td bgcolor="#b2b2b2"><p align='right'>  83.44 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'>  99.39 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#b1b1b1"><p align='right'>  84.51 </td><td bgcolor="#ffffff"><p align='right'>  18.92 </td><td bgcolor="#afafaf"><p align='right'>  85.35 </td><td bgcolor="#9b9b9b"><p align='right'>  99.39 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
</table><PRE>


<b>Similarity table for hits: percentage of chain fitting below 6&#197 RMSD</b>
(Cells are shaded according to similarity starting at 50%)
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  35.56 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9d9d9d"><p align='right'>  98.59 </td><td bgcolor="#9d9d9d"><p align='right'>  98.59 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#ffffff"><p align='right'>  35.56 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  29.91 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  29.91 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#9d9d9d"><p align='right'>  98.59 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#9d9d9d"><p align='right'>  98.59 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
</table><PRE>




<b>ALIGNMENTS</b>

<a name='hit1'><b>--> hit 1</b> <a href='models_LP9_redconf/model_all001.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 34.16% helical, 22.98% extended, 42.86% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BUS' target=_blank>3BUS_A</a> METHYLTRANSFERASE;                                      -17.220    0     0  87.12%  19.72%   OPTM     -438.777     -358.270        2.397        0.622        0.587        0.527        0.913     22.930     -49.8     1.420    46.000     120.5     4.690    52.500    -135.0     1.950    10.540     422.6     0.270     2.240     -50.7     1.090     7.390     429.1     2.230     5.980     -40.5     1.150     9.370     242.4    13.270    14.960     -66.4    -1.000    13.000     -60.0    -1.000    16.000    -118.0     3.600    90.500     162.5    15.720   101.000    -153.0     0.220    15.000      57.0     6.360    41.000    -114.0    19.220     -17.2     4.100      -2.1    19.220     -17.2    0.83  0.94  0.93

alignment source: OPTM
            -------------          HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHH
Query:    1 MSYVPHVPYVPTP----------EKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALA   70
Sbjct:    2 WGENLHFGYWEDAGADVSVDDATDRLTDEMIALLDVRSGDRVLDVGCGIGKPAVRLATARDV-RVTGISISRPQVNQANA   80
            -----------------HHHHHHHHHHHHHHH-------EEEEE-----HHHHHHHHH---E EEEEE--HHHHHHHHHH

            HHHH------EEEEE-----------EEEEEEE--     HHHHHHHHHHHHH-----EEEEE---- ----  HHHHHH
Query:   71 NIEKNGVTGRASIVKGNFFEVDISEATVVTMFLLT-----NVNEMLKPKLEKELKPGTRVVSHEFEI-RGWN--PKEVIK  142
Sbjct:   81 RATAAGLANRVTFSYADAMDLPFEDASFDAVWALESLHHMPDRGRALREMARVLRPGGTVAIADFVLLAPVEGAKKEAVD  160
            HHH------EEEEE------------EEEEEEE--------HHHHHHHHH--EEEEEEEEEEEEEE-----HHHHHHHHH

142 residues (87.12%) of query sequence aligned

<HR>

<a name='hit2'><b>--> hit 2</b> <a href='models_LP9_redconf/model_all002.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 22.69% helical, 16.20% extended, 61.11% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UFK' target=_blank>1UFK_A</a> TT0836 PROTEIN;                                         -15.050    0     0  60.12%  13.51%   OPTM     -187.890     -182.070        1.261        0.436        0.281        0.292        0.560      3.490     -17.4    -0.440     0.000     180.0     8.360    83.500    -162.5    -1.500   -16.280     376.9    -1.750    -0.370     -42.0    -1.870   -13.230     394.4    -1.070     0.880     -32.2     2.420    -9.930     213.7     9.050     2.900     -54.0    -1.000    -4.000     -45.0    -1.000     8.000    -120.0    -0.860   -45.500     263.5    16.460   112.000    -161.0    -0.820     1.000     208.0    -0.010    -3.000     -22.0    17.050     -15.1     4.100      -2.1    17.000     -15.0   -0.13  0.70  0.76

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP9_redconf/model_all002.pdb.log'>here for details</a>
            -EEEEEE--HHHHHHHHHHHHH---      ---EEEEE-----------EEEEEEE--HHHHHHHHHHHHH-----E  
Query:   53 KKAVGVEINDERIREALANIEKNGV------TGRASIVKGNFFEVDISEATVVTMFLLTNVNEMLKPKLEKELKPGTR--  124
Sbjct:    1 MWVYRLKGTLEALDPILPGLFDGGARGLWEREGEVWAFFPA--PVDLPYEGVWEEVG----DEDWLEAWRRDLKPALAPP   74
            -EEEEEE--HHHH---HHHHHH----EEEEE--EEEEEE--  --------EEEE--    HHHHHHHHHHH---EEE--

            EEEE--------HHHHHHE-------EEEEEEE------
Query:  125 VVSHEFEIRGWNPKEVIKVEDGNMNHTVYLYVIGEHKLE  163
Sbjct:   75 FVVLAP-WHTW---------EGAE-----IPLVIEPGMA   98
            EEEE-- ----         ----     EEEE------

98 residues (60.12%) of query sequence aligned

<HR>

<a name='hit3'><b>--> hit 3</b> <a href='models_LP9_redconf/model_all003.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 23.48% helical, 17.81% extended, 58.70% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CJT' target=_blank>3CJT_A</a> RIBOSOMAL PROTEIN L11 METHYLTRANSFERASE;                -14.699    0     1  94.48%  22.93%   OPTM     -452.020     -306.830        2.147        0.604        0.593        0.589        0.948     17.040     -46.4    -0.800    -6.000     201.0     7.570    80.500    -152.0    -2.200    -9.290     451.4    -1.730    -2.260     -30.6    -1.880    -6.300     460.7     0.970     2.570     -24.7    -0.450   -17.630     258.5    15.590    19.600     -67.2   -30.000    -2.000     -46.0   -30.000    16.000    -124.0    -0.460   -29.500     272.5    15.660   110.000    -163.0    -1.530     3.000      88.0     5.590    31.000    -106.0     3.694     -14.7     4.581      -2.3     3.714     -14.7    0.57  0.75  0.76

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP9_redconf/model_all003.pdb.log'>here for details</a>
            -----------HHHHHHHHHH--  -----EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH------
Query:    3 YVPHVPYVPTPEKVVRRMLEIAK--VSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGR   80
Sbjct:   93 IEPGMAFGTGHAETTRLALKALARHLRPGDKVLDLGTGSG-VLAIAAEKLGG-KALGVDIDPMVLPQAEANAKRNGV--R  168
            -----------HHHHHHHHHHHHH-----EEEEE-----H HHHHHHH---E EEEEE-----HHHHHHHHH-----  -

            EEEEE------ -----EEEEEEE--HHHHHHHHHHHHH-----EEEEE--------HHHHHHE-------EEEEEEE--
Query:   81 ASIVKGNFFEV-DISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEIRGWNPKEVIKVEDGNMNHTVYLYVIGE  159
Sbjct:  169 PRFLEGSLEAALPFGPFDLLVANLYAELHAALAPRYREALVPGGRALLTGILKD--RAPLVREAMAGAGFRPLEEAAEGE  246
            EEEE--HHHH-----EEEEEEE--HHHHHHHHHHHHHHEEEEEEEEEEEEE---  HHHHHHHHHH---EEEEEEEE--E

154 residues (94.48%) of query sequence aligned

<HR>

<a name='hit4'><b>--> hit 4</b> <a href='models_LP9_redconf/model_all004.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 35.88% helical, 25.88% extended, 38.24% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXZ' target=_blank>1KXZ_C</a> PRECORRIN-6Y METHYLTRANSFERASE/PUTATIVE                 -14.400    0     0 100.00%  19.63%   OPTM     -531.129     -303.340        3.568        0.584        0.536        0.822        0.935     27.580     -48.4     5.370   133.500    -119.5     9.060   105.000    -186.5    -1.100    -6.100      40.8     0.490    -0.470     -39.4     0.790     4.130      50.9     1.850     1.560     -32.8     6.090    38.330      -2.3    16.780    24.350     -68.6    -1.000    28.000    -109.0    -1.000    33.000    -127.0     8.940   172.500     -94.5    17.800   143.000    -183.0     4.950    48.000     -83.0     9.040    56.000    -133.0    16.400     -14.4     5.000      -3.0    16.520     -14.5    0.73  0.84  0.76

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP9_redconf/model_all004.pdb.log'>here for details</a>
            ----  ---------HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH----
Query:    1 MSYV--PHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVT   78
Sbjct:    5 DEFIKNPSVPG-PTAMEVRCLIMCLAEPGKNDVAVDVGCGTGGVTLELAG--RVRRVYAIDRNPEAISTTEMNLQRHGLG   81
            ----------- --HHHHHHHHHHH------EEEEE-----HHHHHHH--  --EEEEEE--HHHHHHHHHHHHH-----

            --EEEEE-----  ------EEEEEEE--HHHHHHHHHHHHH-----EEEEE---- ----HHHHHHE-------EEEEE
Query:   79 GRASIVKGNFFE--VDISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEI-RGWNPKEVIKVEDGNMNHTVYLY  155
Sbjct:   82 DNVTLMEGDAPEALCKIPDIDIAVVGGSGGELQEILRIIKDKLKPGGRIIVTAILLETKFEAMECLRDLGFDVNITELNI  161
            -EEEEE--HHHHHH----EEEEEE------HHHHHHHHHHHEEEEEEEEEEE--HHHHHHHHHHHHH----EEEEEE--E

            EE------
Query:  156 VIGEHKLE  163
Sbjct:  162 ARGRALDR  169
            EEEE----

163 residues (100.00%) of query sequence aligned

<HR>

<a name='hit5'><b>--> hit 5</b> <a href='models_LP9_redconf/model_all005.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 35.88% helical, 25.88% extended, 38.24% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXZ' target=_blank>1KXZ_A</a> PRECORRIN-6Y METHYLTRANSFERASE/PUTATIVE                 -14.398    0     1 100.00%  19.63%   OPTM     -462.269     -233.550        3.284        0.575        0.560        0.835        0.950     27.760     -48.5     5.440   135.000    -120.5     9.130   106.000    -187.0    -0.780    -2.180      42.6    -0.160    -1.970     -36.9     0.490     3.400      55.7     2.060     2.400     -33.0     5.640    34.570       0.9    17.210    23.930     -68.2   -30.000    28.000    -109.0   -30.000    33.000    -127.0     7.220   146.500     -62.5    15.770   113.000    -154.0     4.830    48.000     -82.0     9.120    56.000    -133.0     5.000     -14.4     4.284      -3.0     4.504     -14.5    0.73  0.84  0.76

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP9_redconf/model_all005.pdb.log'>here for details</a>
            ----  ---------HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH----
Query:    1 MSYV--PHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVT   78
Sbjct:    5 DEFIKNPSVPG-PTAMEVRCLIMCLAEPGKNDVAVDVGCGTGGVTLELAG--RVRRVYAIDRNPEAISTTEMNLQRHGLG   81
            ----------- --HHHHHHHHHHH------EEEEE-----HHHHHHH--  --EEEEEE--HHHHHHHHHHHHH-----

            --EEEEE-----  ------EEEEEEE--HHHHHHHHHHHHH-----EEEEE---- ----HHHHHHE-------EEEEE
Query:   79 GRASIVKGNFFE--VDISEATVVTMFLLTNVNEMLKPKLEKELKPGTRVVSHEFEI-RGWNPKEVIKVEDGNMNHTVYLY  155
Sbjct:   82 DNVTLMEGDAPEALCKIPDIDIAVVGGSGGELQEILRIIKDKLKPGGRIIVTAILLETKFEAMECLRDLGFDVNITELNI  161
            -EEEEE--HHHHHH----EEEEEE------HHHHHHHHHHHEEEEEEEEEEE--HHHHHHHHHHHHH----EEEEEE--E

            EE------
Query:  156 VIGEHKLE  163
Sbjct:  162 ARGRALDR  169
            EEEE----

163 residues (100.00%) of query sequence aligned

<HR>

<b>RESULTS for descision tree LP8_redconf</b>

Phase 1 scoring data: <a href='phase1list.in.1'>input</a> and <a href='phase1.out.1.LP8'>output</a>
Phase 2 scoring data: <a href='phase2.in.123456.LP8_redconf'>input</a> and <a href='phase2.out.123456.LP8_redconf'>output</a>


<a name='hits'><b>HITS:</b>

<a href="legend.htm" target=_blank><i>table legend</i></a>
Model  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                            (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='#hit1'>1  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IM8' target=_blank>1IM8_B</a> YECO;                                                   -41.752    0     0  81.60%  22.56%   OPTM     -492.361     -388.850        2.625        0.673        0.655        0.573        0.946     22.330     -47.4     5.270   119.500     -25.5     6.500    84.000    -155.0     1.370     2.080     251.4    -0.570     0.570     -33.2     3.630    20.470     248.1     2.040    10.080     -27.3     8.100    28.100     115.2    12.530     6.080     -56.7    -1.000     8.000     -77.0    -1.000    -4.000    -109.0     4.930    76.500      66.0    11.300    78.000    -125.0     3.200    20.000     -18.0     8.510    39.000    -121.0    10.100      -8.1     2.390      -0.4    10.100      -8.1   -0.10  0.90  0.75
<a href='#hit2'>2  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3H5I' target=_blank>3H5I_A</a> RESPONSE REGULATOR/SENSORY BOX PROTEIN/GGDEF            -41.821    0     0  71.17%  12.07%   OPTM     -375.968     -166.400        0.826        0.639        0.438        0.865        0.930      3.290     -23.1     0.170     9.500      16.5    -0.570    -1.000     -65.0    -1.670     0.640      23.3    -1.520    -5.850     -28.0    -1.230     2.540      42.4     0.450     0.340     -25.1     3.540    10.300       8.3     9.350     9.430     -46.6    -1.000    -6.000     -52.0    -1.000     3.000     -85.0     0.920    -1.000      38.0    -0.180    -7.000     -40.0     0.960    15.000      12.0     0.860     0.000     -56.0     0.000       2.0     0.000       2.0     0.000       2.0    0.39  0.84  0.00
<a href='#hit3'>3  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HAX' target=_blank>3HAX_D</a> 50S RIBOSOMAL PROTEIN L7AE;                             -41.829    0     0  71.78%  14.53%   OPTM     -428.191     -228.970        0.897        0.645        0.519        0.890        0.919      2.720     -24.0    -0.080   -13.000      37.5     0.010   -13.000     -67.0     0.840    10.110      23.8    -0.590     0.790     -29.4     1.520    10.810      42.4     1.470     4.310     -27.9     3.080    14.880      21.7     9.410     2.850     -43.0    -1.000    -5.000     -47.0    -1.000    -1.000     -79.0     0.630    25.000      57.0     0.180     8.000     -41.0     0.150     6.000      20.0     1.390     5.000     -59.0     0.000       2.0     0.000       2.0     0.000       2.0    0.48  0.58  0.00
<a href='#hit4'>4  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1PXW' target=_blank>1PXW_B</a> LSU RIBOSOMAL PROTEIN L7AE;                             -41.855    0     0  71.17%  14.66%   OPTM     -368.040     -291.700        0.808        0.586        0.508        0.872        0.900      3.100     -23.5    -0.440   -12.000      45.0     0.070   -13.000     -67.5     0.830    10.760      23.6    -0.920    -0.250     -27.4     0.450    10.910      46.8     0.660     1.710     -24.9     1.840    14.710      25.3     8.910     1.520     -42.5    -1.000    -4.000     -47.0    -1.000     1.000     -80.0     0.210    17.000      59.0     0.800    12.000     -46.0    -0.070    10.000      21.0     1.760     2.000     -63.0     0.000       2.0     0.000       2.0     0.000       2.0    0.38  0.61  0.00
<a href='#hit5'>5  </a>    <a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2V74' target=_blank>2V74_F</a> HISTONE-ARGININE METHYLTRANSFERASE CARM1;               -41.887    0     0  69.33%  25.66%   OPTM     -393.352     -218.580        1.323        0.651        0.378        0.322        0.930     19.480     -38.5     1.660    60.000     501.0     6.530    86.500    -152.5    -0.910     1.440    1169.9    -1.160    -4.310     -40.8    -1.070    -5.320    1188.5    -0.870    -8.010     -27.4     0.730     5.310     660.7     9.280    10.010     -52.5    -1.000    15.000      27.0    -1.000    -4.000    -110.0     0.180    25.000     598.0    11.080    60.000    -106.0     0.730    -5.000     252.0     5.110    27.000    -117.0     8.520      -6.5     3.250      -1.2     8.400      -6.4   -0.10  0.82  0.68



<b>RMSD table between hits</b>
(RMSDs are shaded according to similarity, up to 10&#197 )
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#f7f7f7"><p align='right'>   9.27 </td><td bgcolor="#ffffff"><p align='right'>  14.64 </td><td bgcolor="#ffffff"><p align='right'>  14.61 </td><td bgcolor="#bfbfbf"><p align='right'>   3.68 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#f7f7f7"><p align='right'>   9.27 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#ffffff"><p align='right'>  11.62 </td><td bgcolor="#ffffff"><p align='right'>  11.56 </td><td bgcolor="#f1f1f1"><p align='right'>   8.66 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#ffffff"><p align='right'>  14.64 </td><td bgcolor="#ffffff"><p align='right'>  11.62 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#a7a7a7"><p align='right'>   1.27 </td><td bgcolor="#ffffff"><p align='right'>  13.58 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#ffffff"><p align='right'>  14.61 </td><td bgcolor="#ffffff"><p align='right'>  11.56 </td><td bgcolor="#a7a7a7"><p align='right'>   1.27 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td><td bgcolor="#ffffff"><p align='right'>  13.57 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#bfbfbf"><p align='right'>   3.68 </td><td bgcolor="#f1f1f1"><p align='right'>   8.66 </td><td bgcolor="#ffffff"><p align='right'>  13.58 </td><td bgcolor="#ffffff"><p align='right'>  13.57 </td><td bgcolor="#9b9b9b"><p align='right'>   0.00 </td></tr>
</table><PRE>


<b>Similarity table for hits: percentage of chain fitting below 3&#197 RMSD</b>
(Cells are shaded according to similarity starting at 30%)
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  28.83 </td><td bgcolor="#ffffff"><p align='right'>  28.83 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td><td bgcolor="#bebebe"><p align='right'>  75.22 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#ffffff"><p align='right'>  28.83 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  47.41 </td><td bgcolor="#ffffff"><p align='right'>  43.10 </td><td bgcolor="#ffffff"><p align='right'>  28.97 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#ffffff"><p align='right'>  28.83 </td><td bgcolor="#ffffff"><p align='right'>  47.41 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  30.84 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#ffffff"><p align='right'>  29.73 </td><td bgcolor="#ffffff"><p align='right'>  43.10 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#ffffff"><p align='right'>  32.71 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#bebebe"><p align='right'>  75.22 </td><td bgcolor="#ffffff"><p align='right'>  28.97 </td><td bgcolor="#ffffff"><p align='right'>  30.84 </td><td bgcolor="#ffffff"><p align='right'>  32.71 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
</table><PRE>


<b>Similarity table for hits: percentage of chain fitting below 6&#197 RMSD</b>
(Cells are shaded according to similarity starting at 50%)
</PRE><table border='1' style='border-collapse: collapse' bordercolor='#000000'><tr><td> </td><td><p align='right'> 1 </td><td><p align='right'> 2 </td><td><p align='right'> 3 </td><td><p align='right'> 4 </td><td><p align='right'> 5 </td></tr>
<tr><td><p align='right'> 1 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#eaeaea"><p align='right'>  60.36 </td><td bgcolor="#e3e3e3"><p align='right'>  63.96 </td><td bgcolor="#e4e4e4"><p align='right'>  63.06 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
<tr><td><p align='right'> 2 </td><td bgcolor="#eaeaea"><p align='right'>  60.36 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#e5e5e5"><p align='right'>  62.93 </td><td bgcolor="#e3e3e3"><p align='right'>  63.79 </td><td bgcolor="#e7e7e7"><p align='right'>  61.68 </td></tr>
<tr><td><p align='right'> 3 </td><td bgcolor="#e3e3e3"><p align='right'>  63.96 </td><td bgcolor="#e5e5e5"><p align='right'>  62.93 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#e5e5e5"><p align='right'>  62.62 </td></tr>
<tr><td><p align='right'> 4 </td><td bgcolor="#e4e4e4"><p align='right'>  63.06 </td><td bgcolor="#e3e3e3"><p align='right'>  63.79 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#e3e3e3"><p align='right'>  63.55 </td></tr>
<tr><td><p align='right'> 5 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td><td bgcolor="#e7e7e7"><p align='right'>  61.68 </td><td bgcolor="#e5e5e5"><p align='right'>  62.62 </td><td bgcolor="#e3e3e3"><p align='right'>  63.55 </td><td bgcolor="#9b9b9b"><p align='right'> 100.00 </td></tr>
</table><PRE>




<b>ALIGNMENTS</b>

<a name='hit1'><b>--> hit 1</b> <a href='models_LP8_redconf/model_all001.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 35.62% helical, 23.12% extended, 41.25% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IM8' target=_blank>1IM8_B</a> YECO;                                                   -41.752    0     0  81.60%  22.56%   OPTM     -492.361     -388.850        2.625        0.673        0.655        0.573        0.946     22.330     -47.4     5.270   119.500     -25.5     6.500    84.000    -155.0     1.370     2.080     251.4    -0.570     0.570     -33.2     3.630    20.470     248.1     2.040    10.080     -27.3     8.100    28.100     115.2    12.530     6.080     -56.7    -1.000     8.000     -77.0    -1.000    -4.000    -109.0     4.930    76.500      66.0    11.300    78.000    -125.0     3.200    20.000     -18.0     8.510    39.000    -121.0    10.100      -8.1     2.390      -0.4    10.100      -8.1   -0.10  0.90  0.75

alignment source: OPTM
            ----HHHHHHHHHH--  -----EEEEE----HHHHHHHHHH----  EEEEEE--HHHHHHHHHHHHH------EEEEE
Query:   10 VPTPEKVVRRMLEIAK--VSQDDIVYDLGCGDGRIIITAAKDFNVK--KAVGVEINDERIREALANIEKNGVTGRASIVK   85
Sbjct:   17 VPGYSNIITAIGMLAERFVTADSNVYDLGCSRGAATLSARRNINQPNVKIIGIDNSQPMVERCRQHIAAYHSEIPVEILC   96
            --HHHHHHHHHHHHHHH----EEEEEE-----HHHHHHH-------EEEEEEE--HHHHHHHHHHHH-------EEEEE-

            -----------EEEEEEE--     HHHHHHHHHHHHH--- --EEEEE--------HHHHHH
Query:   86 GNFFEVDISEATVVTMFLLT-----NVNEMLKPKLEKELKP-GTRVVSHEFEIRGWNPKEVIK  142
Sbjct:   97 NDIRHVEIKNASMVILNFTLQFLPPEDRIALLTKIYEGLNPNGVLVLSEKFRFEDTKINHLLI  159
            ---------EEEEEEE-------HHHHHHHHHHHHHHEEEEEEEEEEEE-----HHHHHHHHH

133 residues (81.60%) of query sequence aligned

<HR>

<a name='hit2'><b>--> hit 2</b> <a href='models_LP8_redconf/model_all002.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 39.50% helical, 21.85% extended, 38.66% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3H5I' target=_blank>3H5I_A</a> RESPONSE REGULATOR/SENSORY BOX PROTEIN/GGDEF            -41.821    0     0  71.17%  12.07%   OPTM     -375.968     -166.400        0.826        0.639        0.438        0.865        0.930      3.290     -23.1     0.170     9.500      16.5    -0.570    -1.000     -65.0    -1.670     0.640      23.3    -1.520    -5.850     -28.0    -1.230     2.540      42.4     0.450     0.340     -25.1     3.540    10.300       8.3     9.350     9.430     -46.6    -1.000    -6.000     -52.0    -1.000     3.000     -85.0     0.920    -1.000      38.0    -0.180    -7.000     -40.0     0.960    15.000      12.0     0.860     0.000     -56.0     0.000       2.0     0.000       2.0     0.000       2.0    0.39  0.84  0.00

alignment source: OPTM
            ---------HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--H    HHHHHHHHHHHH------
Query:    5 PHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEIND----ERIREALANIEKNGVTGR   80
Sbjct:    1 KKILIVEDSKFQAKTIANILNKYGYTVEIALT---GEAAVEKVSGGWYPDLILMDIELGEGMDGVQTALAIQQISEL--P   75
            -EEEEE---HHHHHHHHHHHHH---EEEEE--   HHHHHHHHH------EEEEE--------HHHHHHHHHHH---  -

            EEEEE--  ----- ----EEEEEEE--HHHHHHHHHHHHH--
Query:   81 ASIVKGN--FFEVD-ISEATVVTMFLLTNVNEMLKPKLEKELK  120
Sbjct:   76 VVFLTAHTEPAVVEKIRSVTAYGYVMKSATEQVLITIVEMALR  118
            EEEEE---------------EEEEEE----HHHHHHHHHHHHH

116 residues (71.17%) of query sequence aligned

<HR>

<a name='hit3'><b>--> hit 3</b> <a href='models_LP8_redconf/model_all003.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 46.67% helical, 15.83% extended, 37.50% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HAX' target=_blank>3HAX_D</a> 50S RIBOSOMAL PROTEIN L7AE;                             -41.829    0     0  71.78%  14.53%   OPTM     -428.191     -228.970        0.897        0.645        0.519        0.890        0.919      2.720     -24.0    -0.080   -13.000      37.5     0.010   -13.000     -67.0     0.840    10.110      23.8    -0.590     0.790     -29.4     1.520    10.810      42.4     1.470     4.310     -27.9     3.080    14.880      21.7     9.410     2.850     -43.0    -1.000    -5.000     -47.0    -1.000    -1.000     -79.0     0.630    25.000      57.0     0.180     8.000     -41.0     0.150     6.000      20.0     1.390     5.000     -59.0     0.000       2.0     0.000       2.0     0.000       2.0    0.48  0.58  0.00

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP8_redconf/model_all003.pdb.log'>here for details</a>
            ----------HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--H  HHHHHHHHHHHH------E
Query:    4 VPHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEIND--ERIREALANIEKNGVTGRA   81
Sbjct:    1 KPSYVKFEVPKELAEKALQAVEIARDTGKIRKG---TNETTKAVERGQAKLVIIAEDVDPEEIVAHLPPLCEEKEIP--Y   75
            ----------HHHHHHHHHHHHHHHHH-EEEE-   HHHHHHHHH-----EEEEE-----------HHHHHHHH---  E

            EEEE--     ---------EEEEEEE--   HHHHHHHHHHHHH--
Query:   82 SIVKGN-----FFEVDISEATVVTMFLLT---NVNEMLKPKLEKELK  120
Sbjct:   76 IYVPSKKELGAAAGIEV-AAASVAIIEPGKARDLVEEIAMKVKELMK  121
            EEE--HHHHHHH----- ---EEEEEE----HHHHHHHHHHHHHHH-

117 residues (71.78%) of query sequence aligned

<HR>

<a name='hit4'><b>--> hit 4</b> <a href='models_LP8_redconf/model_all004.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 45.38% helical, 15.97% extended, 38.66% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1PXW' target=_blank>1PXW_B</a> LSU RIBOSOMAL PROTEIN L7AE;                             -41.855    0     0  71.17%  14.66%   OPTM     -368.040     -291.700        0.808        0.586        0.508        0.872        0.900      3.100     -23.5    -0.440   -12.000      45.0     0.070   -13.000     -67.5     0.830    10.760      23.6    -0.920    -0.250     -27.4     0.450    10.910      46.8     0.660     1.710     -24.9     1.840    14.710      25.3     8.910     1.520     -42.5    -1.000    -4.000     -47.0    -1.000     1.000     -80.0     0.210    17.000      59.0     0.800    12.000     -46.0    -0.070    10.000      21.0     1.760     2.000     -63.0     0.000       2.0     0.000       2.0     0.000       2.0    0.38  0.61  0.00

alignment source: OPTM
 +++ there is a problem with all-atom model, click <a href='models_LP8_redconf/model_all004.pdb.log'>here for details</a>
            ---------HHHHHHHHHH-------EEEEE----HHHHHHHHHH----EEEEEE--H  HHHHHHHHHHHH------EE
Query:    5 PHVPYVPTPEKVVRRMLEIAKVSQDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEIND--ERIREALANIEKNGVTGRAS   82
Sbjct:    1 PSYVKFEVPKELAEKALQAVEIARDTGKIRKG---TNETTKAVERGQAKLVIIAEDVDPEEIVAHLPPLCEEKEIP---Y   74
            ---------HHHHHHHHHHHHHHHHH-EEEE-   HHHHHHHHH-----EEEEE-----------HHHHHHH----   E

            EEE---      --------EEEEEEE--   HHHHHHHHHHHHH--
Query:   83 IVKGNF------FEVDISEATVVTMFLLT---NVNEMLKPKLEKELK  120
Sbjct:   75 IYVPSKKELGAAAGIEV-AAASVAIIEPGKARDLVEEIAMKVRELMK  120
            EEE--HHHHHHH----- ---EEEEEE----HHHHHHHHHHHHHH--

116 residues (71.17%) of query sequence aligned

<HR>

<a name='hit5'><b>--> hit 5</b> <a href='models_LP8_redconf/model_all005.pdb'>[All-atom model (PDB)]</a> <a href='#hits'>[hit list]</a>


  secondary structure: 23.94% helical, 21.13% extended, 54.93% loops/other (query: 31.90% 22.09% 46.01%)
  Template PDB/Molecule                                          score homolog  hits  length   seq.   align    ENE             TE13        old           CGenergy      SIFT2        tma1         tma2     |-------OPTM------| |-------SEQ  (global)-------| |-------SEQ   (local)-------| |-------TRD  (global)-------| |-------TRD   (local)-------| |-------TRS  (global)-------| |-------TRS   (local)-------| |-------TSS  (global)-------| |-------TSS   (local)-------| |-------TSC  (global)-------| |-------TSC   (local)-------| |-------PSM  (global)-------| |-------PSM   (local)-------| |-------SRF  (global)-------| |-------SRF   (local)-------| |-------TBLS------| |-------SBLS------| |-------TBSS------|  sec. str.
                                                                                       (%)    ident.  src.    all-atom      all-atom      score         all-atom     all-atom     all-atom     all-atom   r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy    zscore  r-energy    energy  r-energy    energy  r-energy    energy  r-energy    energy    pos. comp.   tbls_len_ratio
<a href='http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2V74' target=_blank>2V74_F</a> HISTONE-ARGININE METHYLTRANSFERASE CARM1;               -41.887    0     0  69.33%  25.66%   OPTM     -393.352     -218.580        1.323        0.651        0.378        0.322        0.930     19.480     -38.5     1.660    60.000     501.0     6.530    86.500    -152.5    -0.910     1.440    1169.9    -1.160    -4.310     -40.8    -1.070    -5.320    1188.5    -0.870    -8.010     -27.4     0.730     5.310     660.7     9.280    10.010     -52.5    -1.000    15.000      27.0    -1.000    -4.000    -110.0     0.180    25.000     598.0    11.080    60.000    -106.0     0.730    -5.000     252.0     5.110    27.000    -117.0     8.520      -6.5     3.250      -1.2     8.400      -6.4   -0.10  0.82  0.68

alignment source: OPTM
            HHHHHHHHHH---- ---EEEEE----HHHHHHHHHH----EEEEEE--HHHHHHHHHHHHH------EEEEE-------
Query:   14 EKVVRRMLEIAKVS-QDDIVYDLGCGDGRIIITAAKDFNVKKAVGVEINDERIREALANIEKNGVTGRASIVKGNFFEVD   92
Sbjct:   24 TGTYQRAILQNHTDFKDKIVLDVGCGSGILSFFAAQAG-ARKIYAVEAS-TMAQHAEVLVKSNNLTDRIVVIPGKVEEVS  101
            HHHHHHHHH--------EEEEE-----HHHHHHHH--- -EEEEEE--- -HHHHHHHHHH------EEEEE--------

            ----EEEEEEE--         HHHHHHHHHHHHH-----EEE
Query:   93 ISEATVVTMFLLT---------NVNEMLKPKLEKELKPGTRVV  126
Sbjct:  102 LPE-QVDIIISEPMGYMLFNERMLESYLHAK--KYLKPSGNMF  141
            --- EEEEEE-----------HHHHHHH---  -EEEEEEEE-

113 residues (69.33%) of query sequence aligned

<HR>


<b>DONE</b>: Fri Jul 11 14:09:26 2014 EST
</BODY>
</HTML>
